<html>
<head>
<title>PLoS ONE: The Coordinated P53 and Estrogen Receptor Cis-Regulation at an FLT1 Promoter SNP Is Specific to Genotoxic Stress and Estrogenic Compound</title>
<base href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0010236">
</head>
<body>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader">
  <p>Open Access</p>
<p id="articleType">Research Article</p>
</div>

  <h1 rel="dc:type" href="http://purl.org/dc/dcmitype/Text">The Coordinated P53 and Estrogen Receptor Cis-Regulation at an FLT1 Promoter SNP Is Specific to Genotoxic Stress and Estrogenic Compound</h1>
  <div>
<title>Background</title>
<p>Recently, we established that a C&gt;<u>T</u> single nucleotide polymorphism (SNP) in the promoter of the VEGF receptor FLT1 gene generates a ½ site p53 response element (RE-<u>T</u>) that results in p53 responsiveness of the promoter. The transcriptional control required an estrogen receptor (ER) ½ site response element (ERE1) 225 nt upstream to the RE-<u>T</u>.</p>
<title>Methodology/Principal Findings</title>
<p>Here we report the identification of a second ER ½ site (ERE2) located 145 bp downstream of the RE-T and establish that both EREs can impact p53-mediated transactivation of FLT1-T in a manner that is cell type and ER level dependent. Gene reporter assays and ChIP experiments conducted in the breast cancer-derived MCF7 cells revealed that the ERE2 site was sufficient for p53-mediated ERα recruitment and transactivation of the FLT1-T promoter/reporter construct. Surprisingly, unlike the case for other p53 target promoters, p53-mediated transactivation of FLT1-T constructs or expression of the endogenous FLT1 gene, as well as binding of p53 and ER at the promoter constructs, was inducible by doxorubicin but not by 5-fluorouracil. Furthermore, ER activity at FLT1-T was differentially affected by ER ligands, compared to a control TFF1/pS2 ER target promoter. The p53-related transcription factors (TFs) p73 and p63 had no effect on FLT1 transactivation.</p>
<title>Conclusions/Significance</title>
<p>We establish a new dimension to the p53 master regulatory network where p53-mediated transcription from a ½ site RE can be determined by ER binding at one or more <i>cis</i>-acting EREs in manner that is dependent on level of ER protein, the type of ER ligand and the specific p53-inducing agent.</p>
</div>
  <span rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pone.0010236"></span>
  <span></span>
  <span></span>
  <span></span>
  <span></span>
  <span></span>
  <span></span>
  <span></span>

  
<div class="horizontalTabs">
  <ul id="tabsContainer">
<li id="article" class="active"><a href="/article/info%3Adoi%2F10.1371%2Fjournal.pone.0010236;jsessionid=F678A2711196EED21B5C028EE6C93DEE" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0010236;jsessionid=F678A2711196EED21B5C028EE6C93DEE" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0010236;jsessionid=F678A2711196EED21B5C028EE6C93DEE" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0010236;jsessionid=F678A2711196EED21B5C028EE6C93DEE" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenu"> 
    <div class="wrap"> 
      <ul>
<li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" title="Click to turn notes on/off">Hide notes</a>
</li> 
        <li class="discuss icon"> 
            <a href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010236">Make a general comment</a> 
        </li> 
      </ul>
<div id="sectionNavTopBox">
        <p><strong>Jump to</strong></p> 
        <div id="sectionNavTop" class="tools"></div>
      </div>
    </div>
  </div>





<p class="authors"><span rel="dc:creator"><span>Yari Ciribilli</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#equal-contrib">#</a></sup>, <span rel="dc:creator"><span>Virginia Andreotti</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#equal-contrib">#</a></sup>, <span rel="dc:creator"><span>Daniel Menendez</span></span><sup><a href="#aff2">2</a></sup><sup><a href="#equal-contrib">#</a></sup>, <span rel="dc:creator"><span>Jan-Stephan Langen</span></span><sup><a href="#aff3">3</a></sup>, <span rel="dc:creator"><span>Gilbert Schoenfelder</span></span><sup><a href="#aff3">3</a></sup><sup>,</sup><sup><a href="#aff4">4</a></sup>, <span rel="dc:creator"><span>Michael A. Resnick</span></span><sup><a href="#aff2">2</a></sup>, <span rel="dc:creator"><span>Alberto Inga</span></span><sup><a href="#aff1">1</a></sup><sup>,</sup><sup><a href="#aff5">5</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup></p>
<p class="affiliations"><a name="aff1" id="aff1"></a><strong>1</strong> Unit of Molecular Mutagenesis and DNA Repair, National Institute for Cancer Research, IST, Genoa, Italy, <a name="aff2" id="aff2"></a><strong>2</strong> Laboratory of Molecular Genetics, National Institute of Environmental Health Sciences (NIEHS), National Institutes of Health (NIH), Research Triangle Park, North Carolina, United States of America, <a name="aff3" id="aff3"></a><strong>3</strong> Institute of Clinical Pharmacology and Toxicology, Charité-Universitaetsmedizin Berlin, Berlin, Germany, <a name="aff4" id="aff4"></a><strong>4</strong> Institute of Pharmacology and Toxicology, Julius-Maximilians-Universität, Würzburg, Germany, <a name="aff5" id="aff5"></a><strong>5</strong> Centre for Integrative Biology, CIBIO, University of Trento, Trento, Italy</p>
<div class="abstract">
<a id="abstract0" name="abstract0" title="Abstract"></a><h2>Abstract <a href="#top">Top</a>
</h2>
<h3>Background</h3>
<p>Recently, we established that a C&gt;<u>T</u> single nucleotide polymorphism (SNP) in the promoter of the VEGF receptor FLT1 gene generates a ½ site p53 response element (RE-<u>T</u>) that results in p53 responsiveness of the promoter. The transcriptional control required an estrogen receptor (ER) ½ site response element (ERE1) 225 nt upstream to the RE-<u>T</u>.</p>


<h3>Methodology/Principal Findings</h3>
<p>Here we report the identification of a second ER ½ site (ERE2) located 145 bp downstream of the RE-T and establish that both EREs can impact p53-mediated transactivation of FLT1-T in a manner that is cell type and ER level dependent. Gene reporter assays and ChIP experiments conducted in the breast cancer-derived MCF7 cells revealed that the ERE2 site was sufficient for p53-mediated ERα recruitment and transactivation of the FLT1-T promoter/reporter construct. Surprisingly, unlike the case for other p53 target promoters, p53-mediated transactivation of FLT1-T constructs or expression of the endogenous FLT1 gene, as well as binding of p53 and ER at the promoter constructs, was inducible by doxorubicin but not by 5-fluorouracil. Furthermore, ER activity at FLT1-T was differentially affected by ER ligands, compared to a control TFF1/pS2 ER target promoter. The p53-related transcription factors (TFs) p73 and p63 had no effect on FLT1 transactivation.</p>


<h3>Conclusions/Significance</h3>
<p>We establish a new dimension to the p53 master regulatory network where p53-mediated transcription from a ½ site RE can be determined by ER binding at one or more <em>cis</em>-acting EREs in manner that is dependent on level of ER protein, the type of ER ligand and the specific p53-inducing agent.</p>

</div>


<div class="articleinfo">
<p><strong>Citation: </strong>Ciribilli Y, Andreotti V, Menendez D, Langen J-S, Schoenfelder G, et al.  (2010) The Coordinated P53 and Estrogen Receptor Cis-Regulation at an FLT1 Promoter SNP Is Specific to Genotoxic Stress and Estrogenic Compound. PLoS ONE 5(4):
          e10236.
            doi:10.1371/journal.pone.0010236</p>
<p><strong>Editor: </strong>Laszlo Tora, Institute of Genetics and Molecular and Cellular Biology, France</p>
<p></p>
<p><strong>Received:</strong> September 23, 2009; <strong>Accepted:</strong> March 17, 2010; <strong>Published:</strong> April 21, 2010</p>
<p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p>
<p><strong>Funding:</strong> This work was partially supported by the Italian Association for Cancer Research, AIRC (to AI), intramural research funds from NIEHS project 1 Z01 ES065079 (to DM and MAR) and by the Charité-Universitaetsmedizin Berlin (to GS). YC was supported by a fellowship from the PhD School of Genetics Oncology and Developmental Biology, University of Genoa, Italy. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
<p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p>
<p><a name="cor1"></a>* E-mail: <a href="mailto:inga@science.unitn.it">inga@science.unitn.it</a></p>
<p><a name="equal-contrib"></a>#
              These authors contributed equally to this work.
          </p>
</div>





<div id="section1">
<a id="s1" name="s1" title="Introduction"></a><h3>Introduction <a href="#top">Top</a>
</h3>
<p>The Vascular Endothelial Growth Factor Receptor-1 (VEGFR-1), commonly known as FLT1, is a high affinity VEGF receptor belonging to the VEGFR transmembrane receptor tyrosin kinase family, expressed in a variety of cell types, including endothelial cells, hematopoietic stem cells, leucocytes, and osteoblasts <a href="#pone.0010236-Ferrara1">[1]</a>. The FLT1 protein, which possesses higher affinity for VEGFA but weak tyrosine kinase activity, or none at all in the case of the soluble form (sFLT1), can act as inhibitory or decoy <a href="#pone.0010236-Autiero1">[2]</a> to the FLK1/VEGFR-2 receptor. The latter binds to VEGFA and represents a primary driver of angiogenesis in development and healthy conditions. Phenotypes from animal models are consistent with a negative modulation of vasculogenesis/angiogenesis by FLT1 during development <a href="#pone.0010236-Fischer1">[3]</a>. However, FLT1 can positively regulate angiogenesis in the context of various stress responses and diseases, including cancer. Unlike FLK1, the FLT1 gene can be up-regulated by hypoxia due to the presence of an HIF-1α binding site in the promoter <a href="#pone.0010236-Gerber1">[4]</a>. Moreover, only FLT1can bind to VEGFB and Placental Growth Factor (PlGF) ligands that are overexpressed in pathological conditions and result in activation of intracellular signals <a href="#pone.0010236-Fischer1">[3]</a>, <a href="#pone.0010236-Luttun1">[5]</a>. The efficiency of signal transduction via FLT1 and upon binding to specific ligands can also be enhanced by the cell surface coreceptors neuropilin <a href="#pone.0010236-Soker1">[6]</a>, that in addition to binding the semaphorins during neuronal development can also bind selected VEGF subtypes.</p>
<p>Consistent with a role in pathological angiogenesis, FLT1 can be up-regulated in several tumor types, including prostate, breast, colon and non-small cell lung cancer, lung adenocarcinoma, hepatocellular carcinoma and glioblastoma <a href="#pone.0010236-Andre1">[7]</a>–<a href="#pone.0010236-Lee1">[12]</a>. Notably, targeted reduction of FLT1 activity can inhibit tumor cell growth <a href="#pone.0010236-Heidenreich1">[13]</a>–<a href="#pone.0010236-An1">[15]</a>. FLT1 is also important in tissue-specific metastasis, since bone marrow-derived hematopoietic progenitor cells that express FLT1 appear to be required for the formation of the “pre-metastatic niche” <a href="#pone.0010236-Kaplan1">[16]</a>.</p>
<p>Recently, we established that expression from the FLT1 promoter could be modulated in response to genotoxic stress by concomitant activation of the p53 tumor suppressor and Estrogen Receptors (ERs) <a href="#pone.0010236-Menendez1">[17]</a>. Modest p53-dependent responsiveness of the promoter was related to the presence of a single nucleotide polymorphism (SNP) in the proximal FLT1 promoter <a href="#pone.0010236-Menendez2">[18]</a>. The relatively rare C&gt;T allele results in a ½ site p53 response element (RE) that is necessary but not sufficient for a substantial p53-dependent transcriptional effect. Subsequent investigations showed that ligand-activated Estrogen Receptor acting at a ½ site Estrogen Receptor response element (ERE) located upstream of the p53 RE was required for high level of p53-dependent responsiveness <a href="#pone.0010236-Menendez1">[17]</a>.</p>
<p>The master regulator and tumor suppressor p53, which is one of the most important and studied proteins in the cancer field, is a tetrameric (dimer of dimer proteins) sequence-specific transcription factor able to bind to two copies of a decameric sequence with the RRRCWWGYYY consensus (where R stands for a purine, W for A/T and Y for a pyrimidine) <a href="#pone.0010236-elDeiry1">[19]</a>. Recent results, including our own investigation based on functional or DNA binding assays in cell systems or cell extracts, established that maximal transactivation requires adjacent dimer binding sites <a href="#pone.0010236-Wei1">[20]</a>–<a href="#pone.0010236-Jordan1">[26]</a>. A spacer of a few bases dramatically reduced transactivation. We also established that p53 can stimulate transcription, albeit at a reduced levels, from noncanonical response elements including ½ sites <a href="#pone.0010236-Jordan1">[26]</a> (reviewed in <a href="#pone.0010236-Menendez3">[27]</a>). Deviations from consensus are common among established p53 target sites resulting in a wide range of transactivation potentials. The same sequence-specific requirements that were shown to maximize the transactivation potential from full site REs appear to be valid for the ½ site REs <a href="#pone.0010236-Jordan1">[26]</a>. The C&gt;T SNP in the FLT1 promoter changes a critical mismatch in a p53 ½ site that prevents function into a consensus sequence that provides for weak p53 transactivation <a href="#pone.0010236-Menendez1">[17]</a>.</p>
<p>The ERs belong to a large superfamily of nuclear receptor transcription factors that interact with specific ligands to regulate a variety of cellular pathways <a href="#pone.0010236-Deroo1">[28]</a>. The ligand-induced conformational change enables ER to modulate transcription from EREs directly and through recruitment of cofactors. The ERE consensus sequence comprises two inverted repeats of the <span class="gene">GGTCA</span> pentamer, although EREs often contain nonconsensus bases <a href="#pone.0010236-Hall1">[29]</a>–<a href="#pone.0010236-Hall3">[31]</a>. The repeats are typically separated by a 3 nt-based spacer <a href="#pone.0010236-Driscoll1">[32]</a>, <a href="#pone.0010236-KleinHitpass1">[33]</a>. Many ligands can interact with ER proteins and can impact differentially the transactivation at specific EREs. Acting in cooperation with other transcription factors such as Sp1 <a href="#pone.0010236-Porter1">[34]</a>, ERs can also modulate transcription from ½ site EREs.</p>
<p>Our previous results with the FLT1 promoter established transcriptional cooperation between p53 and ER. The noncanonical nature of the cognate response elements present in the promoter enabled strong responsiveness only upon concomitant activation of both transcription factors. In this study these findings have been expanded to an ER positive, p53 wt breast cancer-derived cell model in order to better understand the cooperative relationship between ER and p53 in <em>cis</em>-regulation of FLT1 expression. We found that ER levels, specific ligands and genotoxic stresses can greatly influence the coordinated regulation of expression of the FLT1-T allele. In a more general sense, these findings indicate that a noncanonical p53 RE can provide a wide opportunity for fine-tuning p53-dependent cellular responses and for integrating signaling-responses to complex environmental perturbations.</p>
</div>

<div id="section2">
<a id="s2" name="s2" title="Results"></a><h3>Results <a href="#top">Top</a>
</h3>
<h4>An additional ERE ½ site contributes to p53/ER responsiveness of the FLT1-T promoter</h4>
<p>We previously established that in addition to the ½ site p53 RE (−677 from the Transcriptional Start Site, TSS) a ½ site ERE located 225 nt upstream of the p53 RE (referred to as ERE1; −902 from TSS) is required for efficient p53-induced transactivation of the FLT1 promoter <a href="#pone.0010236-Menendez1">[17]</a>. The cooperative interaction was mainly studied in the colon adenocarcinoma HCT116 cell line that expresses wild-type p53 protein (referred to as p53<sup>+/+</sup>). This cell line is ERα negative and weakly expresses ERβ. To understand better the cooperativity between p53 and ERs in the regulation of FLT1 promoter and the response to genotoxic stress, we evaluated the role of ERs in the p53-dependent transactivation of the FLT1 using a clone of the p53 wild-type breast adenocarcinoma MCF7 cell line which is positive for ERα and has low ERβ expression. Surprisingly, while in HCT116 cells the disruption of the −902 ERE ½ site (<em>i.e.</em>, “ere1”) was sufficient to nearly abolish the transactivation from a 1 kb FLT1-<u>T</u> promoter construct <a href="#pone.0010236-Menendez1">[17]</a>, there was no impact of the disruption in MCF7 cells (<a href="#pone-0010236-g001">Figure 1A</a>). The FLT1-<u>C</u> promoter construct was equivalent to the empty vector.</p>
<div class="figure">
<a name="pone-0010236-g001" id="pone-0010236-g001" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g001"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g001&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g001"><span>Figure 1. </span></a> <span>Two distinct ER ½ site response elements acting in cis along with estrogen receptors contribute to doxorubicin-induced, p53-dependent transactivation of the FLT1-T promoter.</span></strong></p>
<p>(A) MCF7 cells cultured in normal medium. (C &amp; D). HCT116 p53<sup>+/+</sup> cells. Gene reporter assays results are presented as the average light units and the standard errors of three independent biological replicates. The indicated reporter plasmids were transiently transfected along with the pRL-SV40 control vector. When indicated, the HCT116 p53<sup>+/+</sup> cells were treated with doxorubicin (Doxo “+” ) at 0.3 µg/ml for 16 hours prior to cell lysis. Luciferase activity was measured 48 hours post-transfection and normalized for transfection efficiency. Statistically significant differences are highlighted (* =  p&lt;0.01; ∧ = p&lt;0.05, t-test). (B) Partial sequence of the 1 kb FLT1 proximal promoter. The ERE1, p53 RE and ERE2 sequences are highlighted (uppercase: bases matching the consensus sequences) and their relative position with respect to the Transcriptional Start Site (1) is shown. Also presented is a schematic drawing of the reporter constructs based in the pGL3 promoter vector containing a 1 Kb fragment of the FLT1 promoter. The specific mutations introduced to inactivate the ERE1 and ERE2 sites are indicated (bold lowercase). (C,D) Cells were transiently co-transfected with the empty pGL3 vector or the indicated reporter constructs along with an expression vector leading to overexpression of ERα (C) or ERβ (D). (E) HCT116 p53<sup>−/−</sup> cells were used as a control for the p53-dependence of the combined effect of doxorubicin treatment and ERα or ERβ expression on the transactivation of the FLT1 promoter constructs. (F) Representative western blot showing the increase in p53 protein levels after doxorubicin treatment; β-actin was used as loading control. For each cell line, p53 protein levels were assessed in a duplicate experiment to confirm that the transfection with different reporter plasmids did not impact p53 stabilization. (G) HCT116 p53<sup>−/−</sup> cells were transfected with the pS2/TFF1 ER reporter along with empty or overexpression vectors for ERα or ERβ. H. MCF7 cells cultured in estrogen-depleted media supplemented with 10 nM E2.</p>
<span>doi:10.1371/journal.pone.0010236.g001</span><div class="clearer"></div>
</div>
<p>To investigate the cell-specific impact, we first performed <em>in silico</em> analyses of the FLT1 promoter for transcription factor binding sites using the Genomatix MatInspector <a href="#pone.0010236-Cartharius1">[35]</a> as well as Transcription Element Search System (TESS) software (MatInspector: <a href="http://www.genomatix.de/products/MatInspector">http://www.genomatix.de/products/MatInsp​ector</a>; TESS: University of Pennsylvania Computational Biology and Informatics Biology Laboratory, <a href="http://www.cbil.upenn.edu/tess">http://www.cbil.upenn.edu/tess</a>). This led to the identification of a second putative ERE ½ site (ERE2: <span class="gene">GGTCAgga<u>gcgg</u>C</span>; mismatched based from a consensus full site ERE are underlined) located 145 nt downstream to the p53 RE (<a href="#pone-0010236-g001">Figure 1B</a>). To evaluate the contribution of ERE2 in the p53, ER mediated transactivation we developed luciferase-based reporter plasmids containing 1 kb FLT1-T promoter constructs harboring inactivating mutations at either of the ERE sites (“ere1” or “ere2”) as well as a double mutant (“ere1,2”) (described in details in <a href="#pone-0010236-g001">Figure 1B</a>). Gene reporter assays clearly showed that in MCF7 cells both EREs needed to be inactivated to impair p53-dependent transactivation (<a href="#pone-0010236-g001">Figure 1A</a>).</p>
<p>Next we asked whether the relative activity for ERE1 is strictly dependent on the ½ site context (<span class="gene">GGTCA</span>) or could be affected by the adjacent sequence 3′ to it (<span class="gene">GGTCAgag<u>TcACt</u></span>) which, unlike the case of ERE2, contains some features of an ERE ½ site in opposite orientation and correctly spaced from ERE1 although there are mismatches at two critical positions (lowercase) <a href="#pone.0010236-Hall2">[30]</a>, <a href="#pone.0010236-Gewirth1">[36]</a>. We constructed a “pure” ERE1 ½ site by site-directed mutagenesis (referred to as “puERE”, where the remaining consensus ERE bases were changed -T<u>c</u>AC<u>t</u> to T<u>cctt</u>- see <a href="#pone.0010236.s003">Table S1</a> and <a href="#pone-0010236-g001">Figure 1B</a>). This modification did not impact the induction of transactivation in MCF7 cells (<a href="#pone-0010236-g001">Figure 1A</a>).</p>
<p>We then addressed whether the difference in responsiveness seen between HCT116 <a href="#pone.0010236-Menendez1">[17]</a> and MCF7 could be related to intracellular levels of ER proteins. Ectopic overexpression of either ERα or ERβ in HCT116 along with the various FLT1-T constructs and doxorubicin treatment resulted in all but the ere1,2 double mutant being responsive (<a href="#pone-0010236-g001">Figure 1C and 1D</a>). ERβ overexpression resulted in higher relative transactivation of FLT1-T in response to doxorubicin treatment, suggesting an important role for this receptor in the cooperation with p53. However, unlike the case of ERα overexpression, a partial defect associated with the individual ere1 and ere2 constructs was still apparent upon ectopic expression of ERβ. Notably, disruption of ERE2 alone did not completely abolish the responsiveness to doxorubicin even without ER ectopic expression, possibly related to the difference in sequence between ERE1 and ERE2 and their relative closeness to the ER consensus. Consistent with this, the responsiveness to doxorubicin of the puERE construct was significantly lower compared to FLT1-T in HCT116 cells (not shown). Surprisingly, the empty vector was inducible by doxorubicin when the ERs were overexpressed in HCT116 cells (especially with ERβ). An even higher induction of the empty vector by doxorubicin was observed in MCF7 cells (<a href="#pone-0010236-g001">Figure 1A</a>). This might in part be due to the presence in close proximity of both a p53 RE and an ERE in the pGL3 promoter plasmid as predicted by the Gemomatix MathInspector (not shown) and could thus be influenced by the expression of p53 and ER proteins. The p53-dependence of the doxorubicin treatment on transactivation of the FLT1-T constructs was confirmed with a p53-null HCT116-derived cell line (<a href="#pone-0010236-g001">Figure 1E</a>). In the absence of p53, the overexpression of either ERα or ERβ did not lead to any further differential stimulation of the FLT1 reporter constructs although a higher basal level with just the plasmid backbone was observed independent from the SNP status or the ERE sequences (<a href="#pone-0010236-g001">Figure 1E</a>). Doxorubicin treatment resulted in p53 protein stabilization in both p53 positive cell lines and was not affected by transfection with the reporter constructs (<a href="#pone-0010236-g001">Figure 1F</a>). Unlike the FLT1-T reporter, the pS2/TFF1 ER reporter construct was highly inducible by overexpressed ERα or ERβ in HCT116 p53<sup>−/−</sup> cells, confirming the activity of the ER proteins in our culture conditions (<a href="#pone-0010236-g001">Figure 1G</a>). The relative impact of disrupting the ERE1 or ERE2 sites was evaluated in MCF7 cells also in estrogen-reduced culture conditions supplemented with 10 nM estradiol and treated or not with doxorubicin (<a href="#pone-0010236-g001">Figure 1H</a>). Interestingly, in these conditions only the disruption of one ERE was sufficient to bring the doxorubicin responsiveness down to the level of the empty vector. While the results confirmed that ERE2 is a functional ERE, they also highlighted the strong influence of the experimental conditions in the FLT1 cis-regulation.</p>
<p>The transcriptional cooperation between p53 and ERs was also examined in human endothelial cells derived from dermal microvessels (HMEC) using ectopic expression conditions (<a href="#pone-0010236-g002">Figure 2</a>). While the FLT1-C construct was not responsive to p53 and/or ERs, transcriptional cooperation was observed with the FLT1-T, particularly when both ERα and ERβ were overexpressed together with p53. Consistent with the results in MCF7 and in HCT116 p53<sup>+/+</sup> cells, the FLT1-T-ere1 construct was also inducible by ectopic expression of p53, especially when ERα and ERβ were also expressed ectopically.</p>
<div class="figure">
<a name="pone-0010236-g002" id="pone-0010236-g002" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g002"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g002&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g002"><span>Figure 2. </span></a> <span>p53 and estrogen receptors cooperate in FLT1-T transactivation in HMEC cells.</span></strong></p>
<p>Cells were transiently co-transfected with the FLT1-C, FLT1-T or FLT1-T-ere1 reporter constructs along with expression vectors for p53, ERα and/or ERβ. Relative luciferase activity was determined 24 hours after transfection. Presented are the average relative light units and the standard errors of three biological repeats.</p>
<span>doi:10.1371/journal.pone.0010236.g002</span><div class="clearer"></div>
</div>

<h4>Neither p63 nor p73 can modulate transactivation of the FLT1 promoter construct</h4>
<p>Since the p53-related transcription factors p63 and p73 can modulate p53 functions and activate the transcription of some p53 target genes <a href="#pone.0010236-Levrero1">[37]</a>, <a href="#pone.0010236-Flores1">[38]</a>, we examined the impact at the FLT1 promoter alleles using gene reporter assays and ectopic expression of p53 or of selected p63 and p73 isoforms. We also evaluated the effect of doxorubicin treatment along with the ectopic over-expression.</p>
<p>These experiments were conducted in SaOS2 cells which lack endogenous p53. Ectopic p53 expression resulted in modest transactivation of FLT1-T, consistent with the small amount of p53 expression plasmid that was used for transfections (100 ng/well). While in previous experiments more p53 plasmid was used (500 ng/well) <a href="#pone.0010236-Menendez2">[18]</a>, we chose conditions in which the impact of doxorubicin along with ectopic p53 expression could be assessed. In the absence of p53 the FLT1 promoter constructs exhibited low level transactivation similar to the empty vector (<a href="#pone-0010236-g003">Figure 3A</a>). The treatment with doxorubicin led to a small induction (&lt;1.5 fold). The ectopic expression of p53 resulted in a weak stimulation for all reporter constructs (2.0, 2.6 and 2.3 fold, respectively, for empty vector, FLT-C and FLT1-T). The promoter context had an impact in p53-expressing cells, as FLT1-T but not FLT1-C resulted in significantly higher activity compared to the empty vector (1.6 <em>vs</em> 1.2 fold induction). Doxorubicin treatment in p53-expressing cells led to slightly higher transactivation of the FLT1-C reporter (1.3 fold) and greater transactivation of FLT1-T (2.1 fold). The impact of the SNP on the combination of ectopic p53 expression and doxorubicin treatment was highly significant. The PG13 reporter plasmid, a canonical p53 responsive construct, containing a tandem repeat of a p53 RE <a href="#pone.0010236-Kern1">[39]</a>, exhibited high responsiveness to the ectopic expression of p53 that was further stimulated by the doxorubicin treatment (<a href="#pone-0010236-g003">Figure 3B</a>). Ectopic expression of the p53-related transcription factors p73β or p63γ (corresponding to the spliced isoforms that are transcriptionally more active) failed to modulate the response from the FLT1 promoter, regardless of SNP status, although the PG13 control reporter was induced by both proteins.</p>
<div class="figure">
<a name="pone-0010236-g003" id="pone-0010236-g003" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g003"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g003&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g003"><span>Figure 3. </span></a> <span>The p53-related p73β and p63γ proteins do not transactivate the FLT1 promoter constructs in SaOS2 cells.</span></strong></p>
<p>To avoid any contribution of the endogenous p53 protein, gene reporter assays were performed in a p53-null, osteosarcoma-derived SaOS2 cell line. (A) Cells were transiently co-transfected with the FLT1-C or FLT1-T reporter constructs along with an overexpression vector for p53, p73β or p63γ as well as the pRL-SV40 control vector, followed by doxorubicin (0.3 µg/ml) or mock treatment, according to the schedule described in <a href="#pone-0010236-g001">Figure 1</a>. Presented are the average-fold luciferase induction relative to FLT1-C mock treated and the standard errors of three replicates. Statistically significant differences relating to the impact of p53 expression alone or in combination with doxorubicin treatment are highlighted (* =  p&lt;0.01; <sup>∧</sup> = p&lt;0.05, t-test). (B) The PG13 p53 reporter plasmid was used as a control for comparing the transactivation potential of p53, p73β or p63γ and the effect of the doxorubicin treatment.</p>
<span>doi:10.1371/journal.pone.0010236.g003</span><div class="clearer"></div>
</div>

<h4>Differential impact of doxorubicin and 5-fluorouracil on p53-dependent FLT1-T transactivation</h4>
<p>Next we examined whether the nature of the genotoxic stress that leads to p53 stabilization and activation could impact FLT1-T transactivation using MCF7 cells. Interestingly, the thymidylate synthase inhibitor 5-fluorouracil (5FU), another chemotherapeutic agent commonly used to study p53-mediated responses <a href="#pone.0010236-Sun1">[40]</a>–<a href="#pone.0010236-Kaeser1">[42]</a> did not result in transactivation of the FLT1-T construct, although p53 protein levels as well as p21 were increased comparably by doxorubicin and 5FU with respect to mock treated cells (<a href="#pone-0010236-g004">Figure 4A and 4B</a>). In addition, 5FU treatment did not considerably affect the expression levels of ERα protein (<a href="#pone-0010236-g004">Figure 4B</a>), nor its activity, based on gene reporter assays conducted in estrogen-depleted medium supplemented with 100 nM E2 (<a href="#pone-0010236-g004">Figure 4C</a>). Furthermore, as described below, the endogenous FLT1 gene expression was inducible by doxorubicin but not 5FU.</p>
<div class="figure">
<a name="pone-0010236-g004" id="pone-0010236-g004" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g004"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g004&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g004"><span>Figure 4. </span></a> <span>Differential effect of doxorubicin and 5-fluorouracil treatment on FLT1-T transactivation.</span></strong></p>
<p>(A) MCF7 cells were seeded into 12-well plates and transfected either with empty, FLT1-C or FLT1-T plasmids, along with the pRLSV40 control vector. 24 hours after transfection cells were treated with doxorubicin (0.3 µg/ml) or 5FU (375 µM) or mock treated, as indicated. At 16-hour post-treatment, cells were lysed for the luciferase assays. The relative averages and standard errors of at least three independent replicates are presented. (B) Western blot analysis of the p53 (DO-1 antibody), p21 (C-19 antibody) and ERα (H-184 antibody) proteins after treatment with doxorubicin and 5FU. 30 ug of total proteins were loaded in each lane. Actin (C-11 antibody) was used as loading control. All antibodies were obtained from Santa Cruz. The bands above actin are due to a residual p53 signal caused by incomplete removal of the primary Ab against p53 while re-probing for actin. (C) MCF7 cells cultured in estrogen-depleted medium for 72 hrs, were seeded into 24-well plates and transfected with empty or pS2/TFF1 reporter vector. 24 hours after transfection cells were treated with doxorubicin (0.3 µg/ml) or 5FU (375 µM) and/or with the addition of 100 nM E2. At 16-hour post-treatment, cells were lysed for the luciferase assays. The relative averages and standard errors of at least three independent replicates are presented. E2 led to a significant induction (* =  p&lt;0.01, t-test) of the pS2/TFF1 reporter that was not impacted by doxo or 5FU treatment.</p>
<span>doi:10.1371/journal.pone.0010236.g004</span><div class="clearer"></div>
</div>

<h4>Differential impact of doxorubicin and 5FU treatment on the recruitment of p53 at the RE-T and the role of ERE1 and ERE2</h4>
<p>We previously demonstrated using HCT116 p53<sup>+/+</sup> and p53-transfected SaOS2 cells that the ERα or ERβ could bind only the FLT1-T promoter construct in the presence of p53 suggesting that p53 binding was required for ER binding <a href="#pone.0010236-Menendez1">[17]</a>. These findings have been extended to the impact of ERE1 and of the double ERE1, ERE2 mutations on occupancy at EREs and p53 REs using MCF7 cells transfected with the various FLT1 promoter constructs. Specific primers for amplifying the ERE1, ERE2 and the p53 RE sites on plasmid reporters were developed (<a href="#pone-0010236-g005">Figure 5A</a> and <a href="#pone.0010236.s004">Table S2</a>), as well as sonication conditions that enabled us to evaluate the distinct contribution of ERE1 and ERE2. The impact of doxorubicin and 5FU treatments were also compared (<a href="#pone-0010236-g005">Figure 5</a>). First we tested the ability of p53 and ERα to be recruited to a canonical target promoter site (p21 and TFF1, respectively). The different DNA damaging agents resulted in enhanced p53 occupancy at the p21 promoter, consistent with other reports in the literature <a href="#pone.0010236-Espinosa1">[43]</a>, while those same treatments appeared to similarly reduce ER occupancy at the pS2 control target (<a href="#pone.0010236.s001">Figure S1A and S1B</a>). While doxorubicin-treatment led to a significant increase in p53 occupancy levels at the p53 RE-T within the FLT1 promoter fragment, treatment with 5FU did not induce p53 occupancy at the site, consistent with the failure of the 5FU-dependent FLT1-T transactivation (<a href="#pone-0010236-g005">Figure 5B &amp; 5C</a>). Doxorubicin treatment also resulted in ERα occupancy detectable with the primers both for the ERE1 and ERE2 sites (<a href="#pone-0010236-g005">Figure 5B</a>, top panel; 5C, left panel). Inactivation of the ERE1 site did not affect p53 occupancy, but the inactivation of both ERE1 and ERE2 greatly reduced p53 occupancy, consistent with a cooperative <em>cis</em> interaction between p53 and ER (<a href="#pone-0010236-g005">Figure 5B</a>, lower panel; 5C center and right panel). Mutation of the ERE1 site abolished ERα recruitment at the site, but doxorubicin-enhanced occupancy was still detectable at the ERE2 site. The double ere1,2 mutation completely abrogated ERα occupancy. Overall, these results support a distinct role for p53-inducing cellular treatment that appears related to the specific nature of the <em>cis-</em>element sequences in FLT1. Furthermore they indicate that ERE2 is an active ER ½ site and suggest a contribution of ER proteins on the stability/recruitment of p53 at the promoter.</p>
<div class="figure">
<a name="pone-0010236-g005" id="pone-0010236-g005" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g005"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g005&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g005"><span>Figure 5. </span></a> <span>The ERE2 site is important in the recruitment of p53 and ERα on FLT1-T promoter constructs transfected in MCF7 cells.</span></strong></p>
<p>(A) Schematic drawing of the annealing positions of the PCR primers utilized in the ChIP experiments to probe the individual response elements. (B, top panels) representative PCR results that evaluate the occupancy of p53 or ERα after mock, doxorubicin and 5FU treatments obtained with primers specific for the p53 RE, ERE1 and ERE1 region. (B, lower panels) same as in top panel except that the indicated FLT1-T mutant constructs were transfected. ChIP PCR reactions for p53RE, ERE1 and ERE2 were performed with the single mutant ere1 or the double mutant ere1,2 within the pGL3 based constructs after ChIP with p53 and ERα antibodies, respectively. PCR results obtained with input DNA and with IP material obtained from a negative control IgG antibody are also shown. (C) Fold change in site occupancy measured using real time PCR. Data are presented following the same order as in (A). The antibody used for the ChIP experiment was targeted at p53 for the reported analysis at the p53RE site and at ERα for the analysis at the ERE1 or ERE2 sites. Bars represent the average fold change in occupancy and standard deviations of three technical replicates.</p>
<span>doi:10.1371/journal.pone.0010236.g005</span><div class="clearer"></div>
</div>

<h4>ER ligands differentially impact FLT1-T transactivation</h4>
<p>The ER activity at FLT1-T <a href="#pone.0010236-Menendez1">[17]</a> is ligand dependent. We determined if ER ligands differ in their impact on FLT1-T transactivation. Using luciferase assays, the transiently transfected MCF7 cells were treated with several compounds chosen for their established or proposed estrogen-like activity along with doxorubicin. The responses with the pS2/TFF1 ER target promoter were compared to those on FLT1-T (<a href="#pone-0010236-g006">Figure 6A and 6B</a>). Doses were chosen that had similar effects with the pS2/TFF1 ER target promoter. Contrary to the effect on TFF1, genistein, 2-methoxyestradiol and bisphenol A appeared to be more effective than 17β-estradiol (E2) in enhancing doxorubicin-induced FLT1-T transactivation. 2-methoxyestradiol (2Me-OE2) was used based on a previous report of an agonist effect in MCF7 <a href="#pone.0010236-Newman1">[44]</a>, <a href="#pone.0010236-Sutherland1">[45]</a>. Other reports have instead concluded that this molecule is not capable to engage the ERs <a href="#pone.0010236-LaVallee1">[46]</a>. We used different concentration of 2ME-OE2 and examined the impact on the pS2/TFF1 reporter. Interestingly, unlike E2 which was active at all concentrations tested (1, 10, 100 nM), 2Me-OE2 led to reporter transactivation only at the 100 nM concentration. This agonist effect was abolished by the addition of the ICI-182,780 ER antagonist. Zearalenone had a similar impact relative to E2 on the two reporters while 2-methoxyextrone (EI) had a little impact on FLT1-T and nonylphenol was inactive. None of the ligands had any impact on FLT1-T transactivation in the absence of doxorubicin treatment confirming our previous results that FLT1 promoter is not responsive to the ERs alone <a href="#pone.0010236-Menendez1">[17]</a>. The FLT1-T construct was also not responsive to 5FU (less than two fold compared with a 13 fold induction by doxorubicin) (<a href="#pone-0010236-g006">Figure 6B</a>). Consistent with the results presented in <a href="#pone-0010236-g001">Figure 1H</a>, disruption of the ERE1 site abolished doxo responsiveness in estrogen depleted medium. Addition of 100 nM E2 or Genistein did not lead to any induction. The FLT1-C construct was not responsive. The induction observed after genotoxic treatment was equivalent with the empty vector (data not shown; see <a href="#pone-0010236-g001">Figure 1</a>). <a href="#pone.0010236-Menendez1">[17]</a></p>
<div class="figure">
<a name="pone-0010236-g006" id="pone-0010236-g006" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g006"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g006&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g006"><span>Figure 6. </span></a> <span>ER ligand-specific effects on FLT1-T transactivation in doxorubicin treated MCF7 cells.</span></strong></p>
<p>Cells were cultured in estrogen-depleted medium and serum, seeded into 12-well plates and co-transfected with the empty luciferase vector, the pS2/TFF1 ER reporter vector (A) or FLT1-C, FLT1-T or FLT1-Tere1 constructs (B) along with the pRL-SV40 control vector. Cells were incubated with the various ER ligands at the indicated concentrations. Where indicated, cells were also treated with doxorubicin (0.3 µg/ml) or 5FU (375 µM). Luciferase assays were performed on total protein extracts prepared 48 hours post-transfection (16 hours after DNA-damaging treatment, 10 hours after ligand addition). The histograms represent the average light units and the standard errors of three independent biological replicates. E2 = 17-β-estradiol; 2-MeO-E2  = 2-methoxyestradiol; EI = 2-methoxyestrone.</p>
<span>doi:10.1371/journal.pone.0010236.g006</span><div class="clearer"></div>
</div>

<h4>p53-activating agents differentially impact the endogenous FLT1 gene expression</h4>
<p>Using real time RT-PCR approach we quantified endogenous mRNA expression levels of the <em>FLT1</em> gene after different genotoxic treatments. The p53 positive HCT116 p53<sup>+/+</sup> and neuroblastoma-derived GIMEN cell lines were used because they were found to be heterozygous for the C&gt;T SNP in the FLT1 promoter. The MCF7 (C/C) cells were used as controls. The relative mRNA levels for the p53 target gene p21 were measured in comparison with those of FLT1. The basal levels of <em>FLT1</em> mRNA varied considerably among the cell lines before treatment and were nearly undetectable in GIMEN, very low in HCT116, but significantly higher in MCF7 cells (data not shown). The <em>FLT1</em> mRNA was strongly induced by doxorubicin in GIMEN and HCT116, but not in MCF7 cells, while p21 was clearly induced in all three (<a href="#pone-0010236-g007">Figure 7A</a>). Transfected FLT1-T reporter constructs were inducible in the MCF7 cells (<a href="#pone-0010236-g001">Figure 1A</a>), demonstrating the potential for combined p53/ER-mediated regulation. Treatment of GIMEN with 5FU led to the induction of p21 expression but <em>FLT1</em> mRNA levels were not changed, consistent with the results obtained in the gene reporter assays (<a href="#pone-0010236-g007">Figure 7B</a>). To study at the endogenous gene level the cooperation between doxorubicin and E2, mRNA was quantified in GIMEN cells culture in estrogen-depleted medium (<a href="#pone-0010236-g007">Figure 7C</a>). Combined treatment with doxorubicin and E2 led to a significant increase in FLT1 mRNA expression compared the effect of doxorubicin alone. On the contrary the combined treatment slightly reduced p21 expression compared to doxorubicin alone. As an additional control we conducted real time RT-PCR analysis of HMEC cells that are homozygous (C/C) for the SNP in the FLT1 promoter. p53 and/or ERα and/or ERβ were ectopically expressed in these cells that were also treated with ER ligands (estradiol and diethylstilbestrol) or with an ER antagonist (ICI 182,780). Unlike the results with the FLT1-T reporter constructs (<a href="#pone-0010236-g002">Figure 2</a>), we did not observe any induction of the endogenous FLT1 gene, confirming that the combined p53/ER responsiveness requires the T allele and that the gene is not inducible by ERs alone (<a href="#pone.0010236.s002">Figure S2</a>).</p>
<div class="figure">
<a name="pone-0010236-g007" id="pone-0010236-g007" title="Click for larger image " href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g007"><img border="1" src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0010236.g007&amp;representation=PNG_S" align="left" alt="thumbnail" class="thumbnail"></a><p><strong><a href="/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0010236&amp;imageURI=info:doi/10.1371/journal.pone.0010236.g007"><span>Figure 7. </span></a> <span>Exposure-dependent, differential induction of the endogenous FLT1 and p21 genes.</span></strong></p>
<p>The relative expressions of FLT1 (gray bars) and p53 target gene p21 (black bars) in mock, doxorubicin or 5FU treated cells were compared using an RT-PCR approach in real time. GIMEN (neuroblastoma-derived) and HCT116 (colon adenocarcinoma) are heterozygous (CT) for the promoter SNP at the FLT1 promoter while MCF7 cells are homozygous for the p53 nonresponsive SNP allele (CC). The HCT116 p53<sup>−/−</sup> derivative cell line was used as control for p53-independent effects. GAPDH or B2M expression levels served as normalization controls. Histograms described the average fold-induction relative to the housekeeping gene, calculated using the ΔC<sub>t</sub> method. Error bars correspond to standard errors of at least three replicates. (A) Comparison between GIMEN, HCT116 and MCF7 cells. (B) Differential impact of doxorubicin and 5FU on FLT1 expression in GIMEN cells. Since the basal levels of FLT1 expression were very low (C<sub>t</sub>≥30), the fold of induction may appear somewhat exaggerated. C) Impact of doxorubicin and/or E2 on FLT1 and p21 mRNA expression in GIMEN cells culture in estrogen-depleted medium. The increase in FLT1 mRNA expression by combined doxorubicin and E2 treatment appeared to be statistically significant (<sup>∧</sup> = p&lt;0.05, t-test).</p>
<span>doi:10.1371/journal.pone.0010236.g007</span><div class="clearer"></div>
</div>
</div>

<div id="section3">
<a id="s3" name="s3" title="Discussion"></a><h3>Discussion <a href="#top">Top</a>
</h3>
<p>We have characterized the in <em>cis</em> interaction between p53 and estrogen receptors that can result in transcriptional modulation of the FLT1 promoter containing a p53 responsive SNP in a breast cancer cell model where estrogen responses might be expected to be more significant. Specifically, we investigated <em>cis</em>-acting FLT1 promoter features and explored environmental components, namely genotoxic stress conditions and estrogenic ligand exposure, resulting in the combined activation of p53 and ERs. Importantly, we identified a second functional <em>cis</em>-acting ERE ½ site located 145 nt downstream to the p53 ½ site in the FLT1 promoter. Using site-specific mutagenesis, ectopic expression of ERs, transactivation assays and ChIP studies with the breast cancer-derived MCF7 cells, the colon cancer-derived HCT116 cells and the HMEC cells, we confirmed that this ERE2 can participate in the cooperative interaction between p53 and ERs. The ERE2 can recruit ERα even when the ERE1 sequence is mutated. Disruption of both EREs abrogated ER binding and nearly abolished p53 binding to the p53 RE-T region, consistent with the results of the functional assays. Notably, gene reporter assays, RT-qPCR for the endogenous gene, Western Blot and ChIP experiments consistently indicated that the nature of the genotoxic-stress resulting in p53 activation (<em>i.e.</em>, doxorubicin vs. 5-fluorouracil) dictated the engagement and the function of p53 at the FLT1 promoter.</p>
<p>The exact mechanisms that differentiate the impact of doxorubicin and 5FU on FLT1, including the involvement of other <em>cis</em>-acting sequences, remains to be explored and might be related to distinct p53 post-translational modifications induced by the signaling pathways activated upon treatment with these genotoxic drugs <a href="#pone.0010236-Ju1">[41]</a>, <a href="#pone.0010236-Kaeser2">[47]</a>. Alternatively, the mechanisms may relate to the availability of cofactors or even the combined activation of other sequence-specific transcription factors, as proposed in previous studies designed to interpret stress-dependent differences in the global p53 transcriptional network <a href="#pone.0010236-Wei1">[20]</a>, <a href="#pone.0010236-Riley1">[48]</a>, <a href="#pone.0010236-Espinosa2">[49]</a>, or may be a feature that is unique to a ½ site noncanonical p53 target. Overall, our results indicate that the noncanonical nature of the <em>cis</em>-acting elements that enables p53 and ER cooperation at the FLT1 promoter provides higher potential for adaptive tuning of the transcriptional responses compared to highly responsive p53 target genes, such as p21, and may suggest differences in responsiveness to altered function p53 mutants <a href="#pone.0010236-Menendez3">[27]</a>.</p>
<p>In a recent study <a href="#pone.0010236-Jordan1">[26]</a> we examined p53 transactivation potential using various permutations of ½ site RE sequences, confirming that p53 transactivation from this type of noncanonical regulatory sequence is affected by specific sequence features of the RE, similar to what is observed for full-site (tetrameric) canonical REs. The FLT1 ½ site (<span class="gene">GGACATG<u>CT</u>C</span>) ranked low in the comparison of the transactivation potential of ½ site REs, despite the presence of the CATG in the core sequence. In fact, we noted an unexpected negative impact on transactivation that was associated with the “<u>CT</u>” sequence context in the RE. When a transcriptionally optimized ½ site was used, there was a ~20 fold higher transactivation potential compared to the FLT1 RE <a href="#pone.0010236-Jordan1">[26]</a>. It will be important to determine the effect of varying the intrinsic transactivation potential of a p53 RE on the combinatorial interaction between p53 and the estrogen receptor.</p>
<p>In this work, we also explored the contribution of the p53-related p63 and p73 transcription factors in the transactivation of FLT1 and the cooperation with ER, drawing upon recent reports on the differential DNA binding specificity between p53 and p63 <a href="#pone.0010236-Perez1">[50]</a>, <a href="#pone.0010236-Osada1">[51]</a>. The reported preference of p63 for an “S” base (<em>i.e.</em> a C or G) instead of a “W” in the core p53 RE (CWWG) suggested to us the intriguing possibility that p63 and p73 may exhibit preferential activity towards the FLT1-C allele. However, experiments in SaOS2 cells indicated that neither p63 nor p73 play a role in FLT1 transactivation, regardless of SNP status. This lack of activity could be related to a generally lower DNA binding/transactivation potential of p63 and p73 towards weak p53 REs, consistent with previous observations <a href="#pone.0010236-Jegga1">[52]</a>, or to the fact that the cooperativity with the ERs is achieved by specific cofactors whose interaction with the p53 family of transcription factors is restricted to the p53 protein.</p>
<p>In this study we confirmed that the effect of ER on FLT1 transactivation requires at least one <em>cis-</em>acting ERE and is dependent on the interaction with specific ligands. ERα and ERβ appeared to act similarly in the in <em>cis</em> interaction with p53 <a href="#pone.0010236-Menendez1">[17]</a>, although we noted a stronger impact of ERβ in HCT116 cells. The disruption of a single ERE did not impact doxorubicin responsiveness in the MCF7 cells (<a href="#pone-0010236-g001">Figure 1A</a>) and had a small effect in HCT116 p53+/+ or HMEC cells when ERα and especially ERβ were overexpressed (<a href="#pone-0010236-g001">Figure 1C, D</a> and <a href="#pone-0010236-g002">Figure 2</a>). Surprisingly, when MCF7 cells were cultured in estrogen-reduced medium supplemented with 10 nM E2, disruption of either ERE1 or ERE2 resulted in loss of doxorubicin responsiveness (<a href="#pone-0010236-g001">Figure 1H</a>). Consistent with this, previous experiments developed in the osteosarcoma derived U2OS and SaOS2 cells, showed that disruption of a single ERE (ere1) impaired p53-dependent transactivation even with ectopic expression of ERα or ERβ <a href="#pone.0010236-Menendez1">[17]</a>. ChIP assays in SaOS2 showed that ERE1 disruption did not impair p53 binding at the p53 RE but seemed to affect ER binding <a href="#pone.0010236-Menendez1">[17]</a>, based on the lack of recruitment of the TRA220 subunit of the mediator complex that can bind to ERs <a href="#pone.0010236-Zhang1">[53]</a>. Using MCF7 cells, in this study we have confirmed that p53 binding is required for ER binding, but have also revealed that ERα occupancy can stabilize p53 binding at FLT1-T given that disruption of both EREs strongly reduced p53 occupancy. Taken collectively, our results suggest that, besides ER protein levels, other factors can modulate the p53/ER cooperation at the FLT1 proximal promoter in the context of a doxorubicin response. This is consistent with the genotoxic stress-dependency previously described and the cell type specific effects.</p>
<p>Given that only ½ site EREs are present in the FLT1 promoter and that p53 is required for ER-dependent transcriptional stimulation, we evaluated the impact of different ER ligands on FLT1-T transactivation and compared responses with a typical estrogen-responsive promoter, derived from the pS2/TFF1 gene using MCF7 cells cultured in estrogen-reduced conditions. Previous studies revealed ligand dependencies in target specificity <a href="#pone.0010236-Hall2">[30]</a>. Included in our test were the dietary phytoestrogens genistein and zearalenone and the industrial estrogens <em>bisphenol</em> A and nonylphenol <a href="#pone.0010236-Hall2">[30]</a>. We also tested 2-methoxyestradiol, that was shown to exhibit anti-angiogenic activity and could have agonist activity on ERs at high concentrations <a href="#pone.0010236-Sutherland1">[45]</a> and 2-methoxyestrone another estrogen metabolite that had been investigated for anti-proliferative properties <a href="#pone.0010236-Newman1">[44]</a>. Ligand doses were chosen based on the literature <a href="#pone.0010236-Hall2">[30]</a> and on assays with the pS2/TFF1construct (<a href="#pone-0010236-g006">Figure 6A</a>). While exploratory in nature, our results suggest a differential impact of the various ligands in the FLT1-T promoter context, as compared to pS2/TFF1. In particular, genistein and bisphenol A appeared more active than 17β–estradiol, while nonylphenol and 2-methoxyestrone were not active (<a href="#pone-0010236-g006">Figure 6B</a>). It will be interesting to address the relative impact in the FLT1 promoter motif where the ½ site is replaced by a full-site. Previous studies compared different ER ligands using reporter constructs with different ERE sequences <a href="#pone.0010236-Hall2">[30]</a>, <a href="#pone.0010236-Hall3">[31]</a>. While promoter specific effects were reported for the impact of specific ligands or the relative contribution of ERα and ERβ, the quantitative differences among the impacts of genistein, bisphenol A and nonylphenol relative to 17β–estradiol were smaller compared to what is seen with the FLT1-T promoter construct.</p>
<p>The value of p53 mutant status as an independent tumor prognostic marker was found in large cohorts of sporadic breast cancers <a href="#pone.0010236-Olivier1">[54]</a>. Recently, immunohistochemical and molecular analyses have identified different breast cancer subtypes <a href="#pone.0010236-Brenton1">[55]</a>, <a href="#pone.0010236-Sorlie1">[56]</a>. Among these, the basal-like tumors, clinically very aggressive, are defined by the lack of expression of ER, Progesterone Receptor (PR) as well as epidermal growth factor receptor 2 (HER2) and by a high frequency of p53 mutations. Instead the luminal subtypes are p53 wild type and ER/PR positive and associated with a more favorable prognosis. However, luminal B subtype tumors that are classified based on the reduction or loss of ER/PR expression, in some cases associated with gain of HER2 expression, have a significantly worse outcome (54). Direct molecular links between ER expression, p53 WT or mutant status, and disease outcome still need to be fully established. The transcriptional cooperation we uncovered at FLT1 should be evaluated in this context.</p>
<p>The transcriptional circuit provided by the C&gt;T SNP at the FLT1 promoter, appears to bring an additional angiogenesis gene into the p53 transcriptional network. Previous studies have linked p53 to the modulation of angiogenesis both through transcriptional repression of VEGF <a href="#pone.0010236-Pal1">[57]</a> and induction of the metalloproteinase MMP2 <a href="#pone.0010236-Bian1">[58]</a>, maspin <a href="#pone.0010236-Wang2">[59]</a> and PAI <a href="#pone.0010236-Kunz1">[60]</a>. p53 was also shown to be directly involved in the degradation of collagen, resulting in the production of antiangiogenic peptides <a href="#pone.0010236-Teodoro1">[61]</a>. Other studies have linked p53 to the control of the migratory potential of various cell types, including macrophages <a href="#pone.0010236-Komarova1">[62]</a>, <a href="#pone.0010236-Roger1">[63]</a>. Importantly, ER proteins have also been directly linked to the up-regulation of VEGF <a href="#pone.0010236-Hyder1">[64]</a>. Recent results in the literature have highlighted the potential key role of membrane-bound FLT1 in pathological angiogenesis, especially in the context of cancer growth and spread <a href="#pone.0010236-Fischer1">[3]</a>. In particular, FLT1 is the main receptor for VEGFB and PlGF induced signaling and is over-expressed in several cancer cells <a href="#pone.0010236-Fischer1">[3]</a>. In the case of neuroblastoma, FLT1 expression was linked to chemoresistance, especially in the context of low oxygen tension <a href="#pone.0010236-Das1">[8]</a>. Furthermore, bone-marrow-derived FLT1-positive cells are required for the initial formation of pre-metastatic niches in a model of lung tissue metastasis in nude mice <a href="#pone.0010236-Kaplan1">[16]</a>. The biological consequences of the inclusion of FLT1 among the p53 target genes are, however, difficult to predict and await specific investigations. Furthermore, the complexity of the FLT1 regulation is enhanced by the synthesis of a soluble form of the receptor (sFLT1), deriving from alternative splicing of the FLT1 primary transcript that can exert opposite functions compared to the membrane-bound, full-length form (mFLT1) <a href="#pone.0010236-Ferrara2">[65]</a>–<a href="#pone.0010236-Chappel1">[67]</a>.</p>
<p>Overall, our results have identified <em>cis-</em> and <em>trans-</em> factors resulting in the integration of signaling responses that coordinate allele-specific FLT1 transactivation by p53 and estrogen receptors. This study provides further evidence for the functional role of p53 ½ site response elements in modulating p53 transcriptional responses resulting in concerted regulation of gene targets through cooperation with other sequence-specific TFs. This type of functional interaction could be relevant for the modulation of many yet undiscovered gene targets and contribute to the regulation of p53 transactivation selectivity, tailoring responses to specific cellular stress conditions. In a related study, we have examined the influence of the FLT1 promoter motif on the p53/ER transcriptional cooperation with various p53 response element sequences—both canonical and noncanonical placed within the FLT1 promoter context--. This study (in press) has revealed the generality of the <em>in cis</em> interaction between p53 and ER; furthermore, we showed that the enhanced transactivation can extend to cancer-associated p53 mutations.</p>
</div>

<div id="section4">
<a id="s4" name="s4" title="Materials and Methods"></a><h3>Materials and Methods <a href="#top">Top</a>
</h3>
<h4>Cell lines and culture conditions</h4>
<p>The human breast adenocarcinoma-derived MCF7 cell line (p53 wild type, positive for both ERα and ERβ) was obtained from the InterLab Cell Line Collection bank, ICLC (Genoa, Italy). The metastatic neuroblastoma GIMEN cells (p53 wild type) were obtained from GP Tonini (National Institute for Cancer Research, IST, Genoa, Italy), while the colon adenocarcinoma HCT116 (p53<sup>+/+</sup>) cell line and its p53<sup>−/−</sup> derivative and the osteosarcoma-derived SaOS2 cells (p53-null) were a gift from B. Vogelstein (The Johns Hopkins Kimmel Cancer Center, Baltimore, Maryland, USA). Human endothelial cells from dermal microvessels were derived in the Schoenfelder lab and cultured as previously described. Cells were normally maintained in DMEM or RPMI supplemented with 10% FCS and antibiotics (100 units/ml penicillin plus 100 µg/ml streptomycin). GIMEN medium was supplemented with +1% non-essential amino acids. For the experiments with the ER ligands, cells were instead maintained in estrogen-depleted medium consisting of DMEM or RPMI without Phenol Red (Euroclone, Celbio, Milan, Italy) supplemented with 10% Charcoal-Dextran treated FCS (Hyclone, Celbio, Milan, Italy).</p>


<h4>Plasmids</h4>
<p>Vectors carrying 1 Kb fragments derived from the human FLT1 proximal promoter were cloned in pGL3 promoter backbone (Promega, Milan, Italy) as described previously <a href="#pone.0010236-Menendez1">[17]</a>. FLT1-T refers to the fragment containing the rare C&gt;T SNP. FLT1-C is an equivalent fragment containing the common C allele. Site-directed mutagenesis (Stratagene, Milan, Italy) or a two-round, PCR-based, site-specific mutagenesis approach were performed to mutate the putative Estrogen Receptor Response Elements (EREs) within the FLT1-T promoter construct. The sequences of the primers used for site-specific mutagenesis are presented in <a href="#pone.0010236.s003">Table S1</a>. Mammalian expression plasmids harboring p63 and p73 cDNAs were a gift from G. Blandino (Regina Elena Cancer Institute, Rome, Italy) and M. Levrero (University of Rome “La Sapienza”, Rome, Italy). The PG13 p53 reporter plasmid (obtained from B. Vogelstein, The Johns Hopkins Kimmel Cancer Center, Baltimore, Maryland, USA) <a href="#pone.0010236-Kern1">[39]</a>, was used as a transactivation control. The pS2/TFF1 reporter vector contains 1.3 kb of the proximal promoter of the estrogen-responsive gene TFF1 cloned in the pGL3-basic backbone <a href="#pone.0010236-Menendez4">[68]</a>. All mammalian constructs were extracted from XL1blue <em>E. coli</em> cells using QIAex endofree maxi prep kit, according to the manufacturer's protocol (QIAGEN, Milan, Italy).</p>


<h4>Luciferase Assays in Transient Transfection Experiments</h4>
<p>For each cell line 1-2.5×10<sup>5</sup> or 4-8×10<sup>4</sup> cells were seeded respectively in 12-well plates or 24- well 24 h before the transfection. Cells were transfected using FuGENE 6 (Roche, Milan, Italy) according to manufacturer's instructions at ~80% cell confluence. For both types of plates, 300 ng of the pGL3-FLT1 reporter plasmids or pGL3-TFF1/pS2 ER reporter or PG13 p53 reporter were used. When appropriate, 100 ng of the expression vector for ERα/ERβ or for p53 family members (or a control empty vector) were included. All transfections were normalized for efficiency using 50 ng of the pRLSV40 plasmid, harboring the luciferase gene from <em>Renilla reniformis</em> controlled by a constitutive promoter (Promega, Milan, Italy). The total plasmid DNA amount per well was kept constant at 450 ng adding the empty vector pCMV-NeoBam, when appropriate. 24 h after transfection doxorubicin or 5-fluorouracil (5FU) were added in the medium at the indicated doses. In all experiments, cells were harvested 16 h after drug treatment and luciferase assays were conducted as described previously <a href="#pone.0010236-Menendez4">[68]</a>.</p>


<h4>Western Blot Analysis</h4>
<p>Cell extracts were quantified using the BCA protein assay kit and BSA as a reference standard (Pierce, Celbio, Milan, Italy), and equal amount of proteins were resolved on 7.5% BisTris Acrilamide gels, and transferred to Nitrocellulose or PVDF membranes (GE Healthcare, Milan, Italy) using a Biorad MiniProtean apparatus (Biorad, Milan, Italy). After blotting the membranes were probed with monoclonal (p53 specific: pAb1801 and DO-1, Santa Cruz Biotechnology, Milan, Italy) or polyclonal antibodies (p53: CM-1, Novocastra, Milan, Italy; p21: C-19; ERα: H-184, Santa Cruz Biotechnology, Milan, Italy). The quality as well as the equal loading and transfer of protein blots were determined by Ponceau S staining or using a monoclonal antibody against β-actin (C-11 Santa Cruz Biotechnology, Milan, Italy). The relative Molecular mass (<em>M</em><sub>r</sub>) values of the immunoreactive bands were determined by using molecular weight markers (Fermentas, Milan, Italy). After washing, blots were incubated with the appropriate IgG- horseradish peroxidase conjugated secondary antibody (Santa Cruz Biotechnology, Milan, Italy), and immune complexes were visualized by using ECL plus reagent (GE Healthcare, Milan, Italy).</p>


<h4>Chromatin Immunoprecipitation (ChIP) Assays</h4>
<p>ChIP assays were done as previously described <a href="#pone.0010236-Menendez1">[17]</a> using the ChIP kit (Upstate Biotechnology, Millipore, Lake Placid, NY, USA). Briefly, cells were plated onto 150-mm dishes. After 24 h of treatment with doxorubicin or 5FU, cells were fixed with 1% formaldehyde for 10 min at 37°C and then treated with 125 mM glycine for 5 min. Samples were processed following the manufacturer's instructions. Cell lysates were then sonicated using conditions that enabled us to evaluate the distinct contribution of ERE1 and ERE2 which are less then 400 nt distant. The sonication was done using a Misonix 3000 instrument equipped with a deep cup horn. Samples were sonicated using six cycles of 20 second pulses at power setting 8 with a 40 seconds pause in-between. One microgram of ERα (H-184, Santa Cruz, Biotechnology) and DO-7 p53-specific monoclonal antibodies (BD Biosciences Pharmingen, San Jose, CA, USA) were used per ChIP assay. As a negative control we used mouse or rabbit Ig (Santa Cruz Biotechnology). PCR amplifications were performed on immunoprecipitated chromatin using primers to amplify specific regions in the FLT1 promoter (<a href="#pone.0010236.s004">Table S2</a>). The PCR cycles were as follows: an initial 10 min Taq Gold polymerase at 95°C followed by 40 cycles of 95°C for 15 s and 60°C for 1 min. The PCR products were then run on a 1.8% agarose gel and quantified with IMAGEQUANT V5.1 (Molecular Dynamics-GE, Piscataway, NJ, USA). Alternatively, qPCR in real time was used to quantify the fold change in site occupancy. qPCR reaction was done with 2 µL of each sample and using the Power SYBR® Green PCR Master Mix (Applied Biosystems, Foster City, CA, USA) following the manufacturer's recomendation. To determine the fold change in site occupancy the SuperArray ChIP-qPCR Data Analysis tool was used (SA Biosciences, Frederick, MD, USA).</p>


<h4>Real Time PCR</h4>
<p>For the mRNA expression analyses, MCF7, HCT116 and GIMEN cell lines were seeded onto 100 mm Petri dishes and allowed to reach 70–80% of confluence before treating with different drugs as described in the figures. At least 16 hours after treatment cells were harvested and washed once with PBS. Instead, HMEC cells were cultured in estrogen-depleted medium, transiently transfected with p53 and/or ERs expression vectors and treated with pro- and anti- estrogenic compounds at different concentrations. Total RNA was extracted using the RNeasy Kit (Qiagen, Milan, Italy) according to the manufacturer's instructions. For real-time quantitative PCR, cDNA was generated from 1 µg of RNA by using the AffinityScript cDNA Synthesis Kit (Stratagene, Milan, Italy). Real-time PCR was performed on a RotorGene 3000 thermal cycler (Corbett Life Science, Ancona, Italy) using the 5PRIME MasterMix (Eppendorf, Milan, Italy). Primers and TaqMan probes are presented in <a href="#pone.0010236.s004">Table S2</a>. Relative mRNA quantification was obtained using the Δ<sub>Ct</sub> method, where the Glyceraldehyde 3-phosphate dehydrogenase (GAPDH), the β<sub>2</sub>Microglobulin (B2M) or the β-actin genes served as internal control.</p>

</div>

<div id="section5">
<a id="s5" name="s5" title="Supporting Information"></a><h3>Supporting Information <a href="#top">Top</a>
</h3>
<a name="pone.0010236.s001" id="pone.0010236.s001"></a><p><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0010236.s001">Figure S1. </a></strong></p>
<p>Impact of doxorubicin and 5FU on p53 and ER occupancy at target sites. (A): Doxorubicin and 5FU treatment result in a similar increase of p53 occupancy at the p21 promoter. (B) Doxorubicin and 5FU treatment showed a similar negative effect on ERα occupancy at the TFF1 promoter. Shown are representative PCR results obtained using template DNA retrieved from ChIP experiment conducted in MCF7 cells with the indicated primary antibodies and primers specific for the p53 RE and ERE1 containing regions of the target promoters. The effect of doxorubicin and 5FU were compared. In addition to a p53 specific Ab (DO1) and an ERα Ab (H-184 Santa Cruz) the IgG Ab was used as negative control. PCR of input DNA is also shown. (C) Fold change in site occupancy measured using real time PCR. Data are presented following the same order as in panel A. The antibody used for the ChIP experiment was targeted at p53 for the p21 promoter site and at ERα for pS2/TFF1 promoter site.</p>
<p>(0.21 MB DOC)</p>
<a name="pone.0010236.s002" id="pone.0010236.s002"></a><p><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0010236.s002">Figure S2. </a></strong></p>
<p>Quantification of FLT1 mRNA in response to p53, ER overexpression in HMEC cells. (A) Cells were transfected with expression vectors for p53, ERα or ERβ as indicated. (B) Cells co-transfected with p53, ERα and ERβ were also treated 24 hours after transfection by estrogen ligands (Estradiol, E2; Diethylstilbestrol, DES) at the indicated concentrations (M). When indicated, treatment included a 100-fold excess of the ER antagonist ICI 182,780. For both panels, histograms represent the average fold of induction relative to the beta-actin housekeeping gene, calculated using the ΔC<sub>t</sub> method. Error bars present the standard errors of at least three replicates.</p>
<p>(0.09 MB DOC)</p>
<a name="pone.0010236.s003" id="pone.0010236.s003"></a><p><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0010236.s003">Table S1. </a></strong></p>
<p>Primers for 2-round PCR mutagenesis of 1 Kb FLT1-T constructs and description of method.</p>
<p>(0.05 MB DOC)</p>
<a name="pone.0010236.s004" id="pone.0010236.s004"></a><p><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0010236.s004">Table S2. </a></strong></p>
<p>List of the primers (A), the probes (B) used in the Real Time PCR experiments and (C) the primers used in ChIP experiments.</p>
<p>(0.05 MB DOC)</p>
</div>





<div>
<a id="ack" name="ack" title="Acknowledgments"></a><h3>Acknowledgments <a href="#top">Top</a>
</h3>
<p>We thank Drs. Gilberto Fronza and Paola Menichini for critical comments on the manuscript and Drs. Ilaria Traverso and Alessandra Bisio for assistance with luciferase assays.</p>
</div>
<div class="contributions">
<a id="authcontrib" name="authcontrib" title="Author Contributions"></a><h3>Author Contributions <a href="#top">Top</a>
</h3>
<p><span class="capture-id">Conceived and designed the experiments: DM GS MAR AI. Performed the experiments: YC VA DM JSL GS MAR. Analyzed the data: YC VA DM JSL GS MAR AI. Wrote the paper: YC VA DM GS MAR AI.</span></p>
</div>
<div>
<a id="references" name="references" title="References"></a><h3>References <a href="#top">Top</a>
</h3>
<ol class="references">
<li>
<a name="pone.0010236-Ferrara1" id="pone.0010236-Ferrara1"></a><span class="authors">Ferrara N, Gerber HP, LeCouter J</span> (2003) The biology of VEGF and its receptors. Nat Med  9: 669–676. <a class="find" href="/article/findArticle.action?author=Ferrara&amp;title=The%20biology%20of%20VEGF%20and%20its%20receptors.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Autiero1" id="pone.0010236-Autiero1"></a><span class="authors">Autiero M, Waltenberger J, Communi D, Kranz A, Moons, L, et al. </span> (2003) Role of PlGF in the intra- and intermolecular cross talk between the VEGF receptors Flt1 and Flk1. Nat Med  9: 936–943. <a class="find" href="/article/findArticle.action?author=Autiero&amp;title=Role%20of%20PlGF%20in%20the%20intra-%20and%20intermolecular%20cross%20talk%20between%20the%20VEGF%20receptors%20Flt1%20and%20Flk1.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Fischer1" id="pone.0010236-Fischer1"></a><span class="authors">Fischer C, Mazzone M, Jonckx B, Carmeliet P</span> (2008) FLT1 and its ligands VEGFB and PlGF: drug targets for anti-angiogenic therapy? Nat Rev Cancer  8: 942–956. <a class="find" href="/article/findArticle.action?author=Fischer&amp;title=FLT1%20and%20its%20ligands%20VEGFB%20and%20PlGF:%20drug%20targets%20for%20anti-angiogenic%20therapy?">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Gerber1" id="pone.0010236-Gerber1"></a><span class="authors">Gerber HP, Condorelli F, Park J, Ferrara N</span> (1997) Differential transcriptional regulation of the two vascular endothelial growth factor receptor genes. Flt-1, but not Flk-1/KDR, is up-regulated by hypoxia. J Biol Chem  272: 23659–23667. <a class="find" href="/article/findArticle.action?author=Gerber&amp;title=Differential%20transcriptional%20regulation%20of%20the%20two%20vascular%20endothelial%20growth%20factor%20receptor%20genes.%20Flt-1,%20but%20not%20Flk-1/KDR,%20is%20up-regulated%20by%20hypoxia.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Luttun1" id="pone.0010236-Luttun1"></a><span class="authors">Luttun A, Tjwa M, Carmeliet P</span> (2002) Placental growth factor (PlGF) and its receptor Flt-1 (VEGFR-1): novel therapeutic targets for angiogenic disorders. Ann NY Acad Sci  979: 80–93. <a class="find" href="/article/findArticle.action?author=Luttun&amp;title=Placental%20growth%20factor%20(PlGF)%20and%20its%20receptor%20Flt-1%20(VEGFR-1):%20novel%20therapeutic%20targets%20for%20angiogenic%20disorders.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Soker1" id="pone.0010236-Soker1"></a><span class="authors">Soker S, Takashima S, Miao HQ, Neufeld G, Klagsbrun M</span> (1998) Neuropilin-1 is expressed by endothelial and tumor cells as an isoform-specific receptor for vascular endothelial growth factor. Cell  92: 735–745. <a class="find" href="/article/findArticle.action?author=Soker&amp;title=Neuropilin-1%20is%20expressed%20by%20endothelial%20and%20tumor%20cells%20as%20an%20isoform-specific%20receptor%20for%20vascular%20endothelial%20growth%20factor.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Andre1" id="pone.0010236-Andre1"></a><span class="authors">Andre T, Kotelevets L, Vaillant JC, Coudray AM, Weber L, et al. </span> (2000) Vegf, Vegf-B, Vegf-C and their receptors KDR, FLT-1 and FLT-4 during the neoplastic progression of human colonic mucosa. Int J Cancer  86: 174–181. <a class="find" href="/article/findArticle.action?author=Andre&amp;title=Vegf,%20Vegf-B,%20Vegf-C%20and%20their%20receptors%20KDR,%20FLT-1%20and%20FLT-4%20during%20the%20neoplastic%20progression%20of%20human%20colonic%20mucosa.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Das1" id="pone.0010236-Das1"></a><span class="authors">Das B, Yeger H, Tsuchida R, Torkin R, Gee MF, et al. </span> (2005) A hypoxia-driven vascular endothelial growth factor/Flt1 autocrine loop interacts with hypoxia-inducible factor-1alpha through mitogen-activated protein kinase/extracellular signal-regulated kinase 1/2 pathway in neuroblastoma. Cancer Res  65: 7267–7275. <a class="find" href="/article/findArticle.action?author=Das&amp;title=A%20hypoxia-driven%20vascular%20endothelial%20growth%20factor/Flt1%20autocrine%20loop%20interacts%20with%20hypoxia-inducible%20factor-1alpha%20through%20mitogen-activated%20protein%20kinase/extracellular%20signal-regulated%20kinase%201/2%20pathway%20in%20neuroblastoma.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Fan1" id="pone.0010236-Fan1"></a><span class="authors">Fan F, Wey JS, McCarty MF, Belcheva A, Liu W, et al. </span> (2005) Expression and function of vascular endothelial growth factor receptor-1 on human colorectal cancer cells. Oncogene  24: 2647–2653. <a class="find" href="/article/findArticle.action?author=Fan&amp;title=Expression%20and%20function%20of%20vascular%20endothelial%20growth%20factor%20receptor-1%20on%20human%20colorectal%20cancer%20cells.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Ghanem1" id="pone.0010236-Ghanem1"></a><span class="authors">Ghanem MA, van Steenbrugge GJ, Sudaryo MK, Mathoera RB, Nijman JM, et al. </span> (2003) Expression and prognostic relevance of vascular endothelial growth factor (VEGF) and its receptor (FLT-1) in nephroblastoma. J Clin Pathol  56: 107–113. <a class="find" href="/article/findArticle.action?author=Ghanem&amp;title=Expression%20and%20prognostic%20relevance%20of%20vascular%20endothelial%20growth%20factor%20(VEGF)%20and%20its%20receptor%20(FLT-1)%20in%20nephroblastoma.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Ilhan1" id="pone.0010236-Ilhan1"></a><span class="authors">Ilhan N, Deveci F</span> (2004) Functional significance of vascular endothelial growth factor and its receptor (receptor-1) in various lung cancer types. Clin Biochem  37: 840–845. <a class="find" href="/article/findArticle.action?author=Ilhan&amp;title=Functional%20significance%20of%20vascular%20endothelial%20growth%20factor%20and%20its%20receptor%20(receptor-1)%20in%20various%20lung%20cancer%20types.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Lee1" id="pone.0010236-Lee1"></a><span class="authors">Lee TH, Seng S, Sekine M, Hinton C, Fu Y, et al. </span> (2007) Vascular endothelial growth factor mediates intracrine survival in human breast carcinoma cells through internally expressed VEGFR1/FLT1. PLoS Med  4: e186. <a class="find" href="/article/findArticle.action?author=Lee&amp;title=Vascular%20endothelial%20growth%20factor%20mediates%20intracrine%20survival%20in%20human%20breast%20carcinoma%20cells%20through%20internally%20expressed%20VEGFR1/FLT1.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Heidenreich1" id="pone.0010236-Heidenreich1"></a><span class="authors">Heidenreich R, Machein M, Nicolaus A, Hilbig A, Wild C, et al. </span> (2004) Inhibition of solid tumor growth by gene transfer of VEGF receptor-1 mutants. Int J Cancer  111: 348–357. <a class="find" href="/article/findArticle.action?author=Heidenreich&amp;title=Inhibition%20of%20solid%20tumor%20growth%20by%20gene%20transfer%20of%20VEGF%20receptor-1%20mutants.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Ishizaki1" id="pone.0010236-Ishizaki1"></a><span class="authors">Ishizaki H, Tsunoda T, Wada S, Yamauchi M, Shibuya M, et al. </span> (2006) Inhibition of tumor growth with antiangiogenic cancer vaccine using epitope peptides derived from human vascular endothelial growth factor receptor 1. Clin Cancer Res  12: 5841–5849. <a class="find" href="/article/findArticle.action?author=Ishizaki&amp;title=Inhibition%20of%20tumor%20growth%20with%20antiangiogenic%20cancer%20vaccine%20using%20epitope%20peptides%20derived%20from%20human%20vascular%20endothelial%20growth%20factor%20receptor%201.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-An1" id="pone.0010236-An1"></a><span class="authors">An P, Lei H, Zhang J, Song S, He L, et al. </span> (2004) Suppression of tumor growth and metastasis by a VEGFR-1 antagonizing peptide identified from a phage display library. Int J Cancer  111: 165–173. <a class="find" href="/article/findArticle.action?author=An&amp;title=Suppression%20of%20tumor%20growth%20and%20metastasis%20by%20a%20VEGFR-1%20antagonizing%20peptide%20identified%20from%20a%20phage%20display%20library.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Kaplan1" id="pone.0010236-Kaplan1"></a><span class="authors">Kaplan RN, Riba RR, Zacharoulis S, Bramley AH, Vincent L, et al. </span> (2005) VEGFR1-positive haematopoietic bone marrow progenitors initialte the pre-metastatic niche. Nature  438: 820–827. <a class="find" href="/article/findArticle.action?author=Kaplan&amp;title=VEGFR1-positive%20haematopoietic%20bone%20marrow%20progenitors%20initialte%20the%20pre-metastatic%20niche.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Menendez1" id="pone.0010236-Menendez1"></a><span class="authors">Menendez D, Inga A, Snipe J, Krysiak O, Schonfelder G, et al. </span> (2007) A single-nucleotide polymorphism in a half-binding site creates p53 and estrogen receptor control of vascular endothelial growth factor receptor 1. Mol Cell Biol  27: 2590–2600. <a class="find" href="/article/findArticle.action?author=Menendez&amp;title=A%20single-nucleotide%20polymorphism%20in%20a%20half-binding%20site%20creates%20p53%20and%20estrogen%20receptor%20control%20of%20vascular%20endothelial%20growth%20factor%20receptor%201.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Menendez2" id="pone.0010236-Menendez2"></a><span class="authors">Menendez D, Krysiak O, Inga A, Krysiak B, Resnick MA, et al. </span> (2006) A SNP in the flt-1 promoter integrates the VEGF system into the p53 transcriptional network. Proc Natl Acad Sci USA  103: 1406–1411. <a class="find" href="/article/findArticle.action?author=Menendez&amp;title=A%20SNP%20in%20the%20flt-1%20promoter%20integrates%20the%20VEGF%20system%20into%20the%20p53%20transcriptional%20network.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-elDeiry1" id="pone.0010236-elDeiry1"></a><span class="authors">el-Deiry WS, Kern SE, Pietenpol JA, Kinzler KW, Vogelstein B</span> (1992) Definition of a consensus binding site for p53. Nat Genet  1: 45–49. <a class="find" href="/article/findArticle.action?author=el-Deiry&amp;title=Definition%20of%20a%20consensus%20binding%20site%20for%20p53.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Wei1" id="pone.0010236-Wei1"></a><span class="authors">Wei CL, Wu Q, Vega VB, Chiu KP, Ng P, et al. </span> (2006) A global map of p53 transcription-factor binding sites in the human genome. Cell  124: 207–219. <a class="find" href="/article/findArticle.action?author=Wei&amp;title=A%20global%20map%20of%20p53%20transcription-factor%20binding%20sites%20in%20the%20human%20genome.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Tokino1" id="pone.0010236-Tokino1"></a><span class="authors">Tokino T, Thiagalingam S, el-Deiry WS, Waldman T, Kinzler KW</span> (1994) p53 tagged sites from human genomic DNA. Hum Mol Genet  3: 1537–1542. <a class="find" href="/article/findArticle.action?author=Tokino&amp;title=p53%20tagged%20sites%20from%20human%20genomic%20DNA.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Veprintsev1" id="pone.0010236-Veprintsev1"></a><span class="authors">Veprintsev DB, Fersht AR</span> (2008) Algorithm for prediction of tumour suppressor p53 affinity for binding sites in DNA. Nucleic Acids Res  36: 1589–1598. <a class="find" href="/article/findArticle.action?author=Veprintsev&amp;title=Algorithm%20for%20prediction%20of%20tumour%20suppressor%20p53%20affinity%20for%20binding%20sites%20in%20DNA.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Wang1" id="pone.0010236-Wang1"></a><span class="authors">Wang Y, Schwedes JF, Parks D, Mann K, Tegtmeyer P</span> (1995) Interaction of p53 with its consensus DNA-binding site. Mol Cell Biol  15: 2157–2165. <a class="find" href="/article/findArticle.action?author=Wang&amp;title=Interaction%20of%20p53%20with%20its%20consensus%20DNA-binding%20site.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-McLure1" id="pone.0010236-McLure1"></a><span class="authors">McLure KG, Lee PW</span> (1998) How p53 binds DNA as a tetramer. Embo J  17: 3342–3350. <a class="find" href="/article/findArticle.action?author=McLure&amp;title=How%20p53%20binds%20DNA%20as%20a%20tetramer.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Inga1" id="pone.0010236-Inga1"></a><span class="authors">Inga A, Storici F, Darden TA, Resnick MA</span> (2002) Differential transactivation by the p53 transcription factor is highly dependent on p53 level and promoter target sequence. Mol Cell Biol  22: 8612–8625. <a class="find" href="/article/findArticle.action?author=Inga&amp;title=Differential%20transactivation%20by%20the%20p53%20transcription%20factor%20is%20highly%20dependent%20on%20p53%20level%20and%20promoter%20target%20sequence.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Jordan1" id="pone.0010236-Jordan1"></a><span class="authors">Jordan JJ, Menendez D, Inga A, Nourredine M, Bell D, et al. </span> (2008) Noncanonical DNA motifs as transactivation targets by wild type and mutant p53. PLoS Genet  4: e1000104. <a class="find" href="/article/findArticle.action?author=Jordan&amp;title=Noncanonical%20DNA%20motifs%20as%20transactivation%20targets%20by%20wild%20type%20and%20mutant%20p53.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Menendez3" id="pone.0010236-Menendez3"></a><span class="authors">Menendez D, Inga A, Resnick MA</span> (2009) The expanding universe of p53 targets. Nat Rev Cancer  9: 724–737. <a class="find" href="/article/findArticle.action?author=Menendez&amp;title=The%20expanding%20universe%20of%20p53%20targets.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Deroo1" id="pone.0010236-Deroo1"></a><span class="authors">Deroo BJ, Korach KS</span> (2006) Estrogen receptors and human disease. J Clin Invest  116: 561–570. <a class="find" href="/article/findArticle.action?author=Deroo&amp;title=Estrogen%20receptors%20and%20human%20disease.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Hall1" id="pone.0010236-Hall1"></a><span class="authors">Hall JM, Couse JF, Korach KS</span> (2001) The multifaceted mechanisms of estradiol and estrogen receptor signaling. J Biol Chem  276: 36869–36872. <a class="find" href="/article/findArticle.action?author=Hall&amp;title=The%20multifaceted%20mechanisms%20of%20estradiol%20and%20estrogen%20receptor%20signaling.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Hall2" id="pone.0010236-Hall2"></a><span class="authors">Hall JM, Korach KS</span> (2002) Analysis of the molecular mechanisms of human estrogen receptors alpha and beta reveals differential specificity in target promoter regulation by xenoestrogens. J Biol Chem  277: 44455–44461. <a class="find" href="/article/findArticle.action?author=Hall&amp;title=Analysis%20of%20the%20molecular%20mechanisms%20of%20human%20estrogen%20receptors%20alpha%20and%20beta%20reveals%20differential%20specificity%20in%20target%20promoter%20regulation%20by%20xenoestrogens.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Hall3" id="pone.0010236-Hall3"></a><span class="authors">Hall JM, McDonnell DP, Korach KS</span> (2002) Allosteric regulation of estrogen receptor structure, function, and coactivator recruitment by different estrogen response elements. Mol Endocrinol  16: 469–486. <a class="find" href="/article/findArticle.action?author=Hall&amp;title=Allosteric%20regulation%20of%20estrogen%20receptor%20structure,%20function,%20and%20coactivator%20recruitment%20by%20different%20estrogen%20response%20elements.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Driscoll1" id="pone.0010236-Driscoll1"></a><span class="authors">Driscoll MD, Sathya G, Muyan M, Klinge CM, Hilf R, et al. </span> (1998) Sequence requirements for estrogen receptor binding to estrogen response elements. J Biol Chem  273: 29321–30. <a class="find" href="/article/findArticle.action?author=Driscoll&amp;title=Sequence%20requirements%20for%20estrogen%20receptor%20binding%20to%20estrogen%20response%20elements.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-KleinHitpass1" id="pone.0010236-KleinHitpass1"></a><span class="authors">Klein-Hitpass L, Ryffel GU, Heitlinger E, Cato, AC</span> (1988) A 13 bp palindrome is a functional estrogen responsive element and interacts specifically with estrogen receptor. Nucleic Acids Res  16: 647–663. <a class="find" href="/article/findArticle.action?author=Klein-Hitpass&amp;title=A%2013%20bp%20palindrome%20is%20a%20functional%20estrogen%20responsive%20element%20and%20interacts%20specifically%20with%20estrogen%20receptor.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Porter1" id="pone.0010236-Porter1"></a><span class="authors">Porter W, Saville B, Hoivik D, Safe S</span> (1997) Functional synergy between the transcription factor Sp1 and the estrogen receptor. Mol Endocrinol  11: 1569–1580. <a class="find" href="/article/findArticle.action?author=Porter&amp;title=Functional%20synergy%20between%20the%20transcription%20factor%20Sp1%20and%20the%20estrogen%20receptor.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Cartharius1" id="pone.0010236-Cartharius1"></a><span class="authors">Cartharius K, Frech K, Grote K, Klocke B, Haltmeier M, et al. </span> (2005) MatInspector and beyond: promoter analysis based on transcription factor binding sites. Bioinformatics  21: 2933–2942. <a class="find" href="/article/findArticle.action?author=Cartharius&amp;title=MatInspector%20and%20beyond:%20promoter%20analysis%20based%20on%20transcription%20factor%20binding%20sites.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Gewirth1" id="pone.0010236-Gewirth1"></a><span class="authors">Gewirth DT, Sigler PB</span> (1995) The basis for half-site specificity explored through a non-cognate steroid receptor-DNA complex. Nat Struct Biol  2: 386–394. <a class="find" href="/article/findArticle.action?author=Gewirth&amp;title=The%20basis%20for%20half-site%20specificity%20explored%20through%20a%20non-cognate%20steroid%20receptor-DNA%20complex.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Levrero1" id="pone.0010236-Levrero1"></a><span class="authors">Levrero M, De Laurenzi V, Costanzo A, Gong J, Wang JY, et al. </span> (2000) The p53/p63/p73 family of transcription factors: overlapping and distinct functions. J Cell Sci  113: 1661–1670. <a class="find" href="/article/findArticle.action?author=Levrero&amp;title=The%20p53/p63/p73%20family%20of%20transcription%20factors:%20overlapping%20and%20distinct%20functions.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Flores1" id="pone.0010236-Flores1"></a><span class="authors">Flores ER, Tsai KY, Crowley D, Sengupta S, Yang A, et al. </span> (2002) p63 and p73 are required for p53-dependent apoptosis in response to DNA damage. Nature  416: 560–564. <a class="find" href="/article/findArticle.action?author=Flores&amp;title=p63%20and%20p73%20are%20required%20for%20p53-dependent%20apoptosis%20in%20response%20to%20DNA%20damage.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Kern1" id="pone.0010236-Kern1"></a><span class="authors">Kern SE, Pietenpol JA, Thiagalingam S, Seymour A, Kinzler KW, et al. </span> (1992) Oncogenic forms of p53 inhibit p53-regulated gene expression. Science  256: 827–830. <a class="find" href="/article/findArticle.action?author=Kern&amp;title=Oncogenic%20forms%20of%20p53%20inhibit%20p53-regulated%20gene%20expression.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Sun1" id="pone.0010236-Sun1"></a><span class="authors">Sun XX, Dai MS, Lu H</span> (2007) 5-fluorouracil activation of p53 involves an MDM2-ribosomal protein interaction. J Biol Chem  282: 8052–8059. <a class="find" href="/article/findArticle.action?author=Sun&amp;title=5-fluorouracil%20activation%20of%20p53%20involves%20an%20MDM2-ribosomal%20protein%20interaction.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Ju1" id="pone.0010236-Ju1"></a><span class="authors">Ju J, Schmitz JC, Song B, Kudo K, Chu E</span> (2007) Regulation of p53 expression in response to 5-fluorouracil in human cancer RKO cells. Clin Cancer Res  13: 4245–4251. <a class="find" href="/article/findArticle.action?author=Ju&amp;title=Regulation%20of%20p53%20expression%20in%20response%20to%205-fluorouracil%20in%20human%20cancer%20RKO%20cells.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Kaeser1" id="pone.0010236-Kaeser1"></a><span class="authors">Kaeser MD, Iggo RD</span> (2004) Promoter-specific p53-dependent histone acetylation following DNA damage. Oncogene  23: 4007–4013. <a class="find" href="/article/findArticle.action?author=Kaeser&amp;title=Promoter-specific%20p53-dependent%20histone%20acetylation%20following%20DNA%20damage.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Espinosa1" id="pone.0010236-Espinosa1"></a><span class="authors">Espinosa JM, Verdun RE, Emerson BM</span> (2003) p53 functions through stress- and promoter-specific recruitment of transcription initiation components before and after DNA damage. Mol Cell  12: 1015–1027. <a class="find" href="/article/findArticle.action?author=Espinosa&amp;title=p53%20functions%20through%20stress-%20and%20promoter-specific%20recruitment%20of%20transcription%20initiation%20components%20before%20and%20after%20DNA%20damage.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Newman1" id="pone.0010236-Newman1"></a><span class="authors">Newman SP, Leese MP, Purohit A, James DR, Rennie CE, et al. </span> (2004) Inhibition of in vitro angiogenesis by 2-methoxy- and 2-ethyl-estrogen sulfamates. Int J Cancer  109: 533–540. <a class="find" href="/article/findArticle.action?author=Newman&amp;title=Inhibition%20of%20in%20vitro%20angiogenesis%20by%202-methoxy-%20and%202-ethyl-estrogen%20sulfamates.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Sutherland1" id="pone.0010236-Sutherland1"></a><span class="authors">Sutherland TE, Schuliga M, Harris T, Eckhardt BL, Anderson RL, et al. </span> (2005) 2-methoxyestradiol is an estrogen receptor agonist that supports tumor growth in murine xenograft models of breast cancer. Clin Cancer Res  11: 1722–1732. <a class="find" href="/article/findArticle.action?author=Sutherland&amp;title=2-methoxyestradiol%20is%20an%20estrogen%20receptor%20agonist%20that%20supports%20tumor%20growth%20in%20murine%20xenograft%20models%20of%20breast%20cancer.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-LaVallee1" id="pone.0010236-LaVallee1"></a><span class="authors">LaVallee TM, Zhan XH, Herbstritt CJ, Kough EC, Green SJ, et al. </span> (2002) 2-Methoxyestradiol inhibits proliferation and induces apoptosis independently of estrogen receptors alpha and beta. Cancer Res  62: 3691–3697. <a class="find" href="/article/findArticle.action?author=LaVallee&amp;title=2-Methoxyestradiol%20inhibits%20proliferation%20and%20induces%20apoptosis%20independently%20of%20estrogen%20receptors%20alpha%20and%20beta.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Kaeser2" id="pone.0010236-Kaeser2"></a><span class="authors">Kaeser MD, Pebernard S, Iggo RD</span> (2004) Regulation of p53 stability and function in HCT116 colon cancer cells. J Biol Chem  279: 7598–7605. <a class="find" href="/article/findArticle.action?author=Kaeser&amp;title=Regulation%20of%20p53%20stability%20and%20function%20in%20HCT116%20colon%20cancer%20cells.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Riley1" id="pone.0010236-Riley1"></a><span class="authors">Riley T, Sontag E, Chen P, Levine A</span> (2008) Transcriptional control of human p53-regulated genes. Nat Rev Mol Cell Biol  9: 402–412. <a class="find" href="/article/findArticle.action?author=Riley&amp;title=Transcriptional%20control%20of%20human%20p53-regulated%20genes.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Espinosa2" id="pone.0010236-Espinosa2"></a><span class="authors">Espinosa JM</span> (2008) Mechanisms of regulatory diversity within the p53 transcriptional network. Oncogene  27: 4013–4023. <a class="find" href="/article/findArticle.action?author=Espinosa&amp;title=Mechanisms%20of%20regulatory%20diversity%20within%20the%20p53%20transcriptional%20network.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Perez1" id="pone.0010236-Perez1"></a><span class="authors">Perez CA, Ott J, Mays DJ, Pietenpol JA</span> (2007) p63 consensus DNA-binding site: identification, analysis and application into a p63MH algorithm. Oncogene  26: 7363–7370. <a class="find" href="/article/findArticle.action?author=Perez&amp;title=p63%20consensus%20DNA-binding%20site:%20identification,%20analysis%20and%20application%20into%20a%20p63MH%20algorithm.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Osada1" id="pone.0010236-Osada1"></a><span class="authors">Osada M, Park HL, Nagakawa Y, Yamashita K, Fomenkov A, et al. </span> (2005) Differential recognition of response elements determines target gene specificity for p53 and p63. Mol Cell Biol  25: 6077–6089. <a class="find" href="/article/findArticle.action?author=Osada&amp;title=Differential%20recognition%20of%20response%20elements%20determines%20target%20gene%20specificity%20for%20p53%20and%20p63.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Jegga1" id="pone.0010236-Jegga1"></a><span class="authors">Jegga AG, Inga A, Menendez D, Aronow BJ, Resnick MA</span> (2008) Functional evolution of the p53 regulatory network through its target response elements. Proc Natl Acad Sci USA  105: 944–949. <a class="find" href="/article/findArticle.action?author=Jegga&amp;title=Functional%20evolution%20of%20the%20p53%20regulatory%20network%20through%20its%20target%20response%20elements.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Zhang1" id="pone.0010236-Zhang1"></a><span class="authors">Zhang X, Krutchinsky A, Fukuda A, Chen W, Yamamura S, et al. </span> (2005) MED1/TRAP220 exists predominantly in a TRAP/Mediator subpopulation enriched in RNA polymerase II and is required for ER-mediated transcription. Mol Cell  19: 89–100. <a class="find" href="/article/findArticle.action?author=Zhang&amp;title=MED1/TRAP220%20exists%20predominantly%20in%20a%20TRAP/Mediator%20subpopulation%20enriched%20in%20RNA%20polymerase%20II%20and%20is%20required%20for%20ER-mediated%20transcription.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Olivier1" id="pone.0010236-Olivier1"></a><span class="authors">Olivier M, Langerod A, Carrieri P, Bergh J, Klaar S, et al. </span> (2006) The clinical value of somatic TP53 gene mutations in 1,794 patients with breast cancer. Clin Cancer Res  12: 1157–1167. <a class="find" href="/article/findArticle.action?author=Olivier&amp;title=The%20clinical%20value%20of%20somatic%20TP53%20gene%20mutations%20in%201,794%20patients%20with%20breast%20cancer.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Brenton1" id="pone.0010236-Brenton1"></a><span class="authors">Brenton JD, Carey LA, Ahmed AA, Caldas C</span> (2005) Molecular classification and molecular forecasting of breast cancer: ready for clinical application? J Clin Oncol  23: 7350–7360. <a class="find" href="/article/findArticle.action?author=Brenton&amp;title=Molecular%20classification%20and%20molecular%20forecasting%20of%20breast%20cancer:%20ready%20for%20clinical%20application?">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Sorlie1" id="pone.0010236-Sorlie1"></a><span class="authors">Sorlie T, Tibshirani R, Parker J, Hastie T, Marron JS, et al. </span> (2003) Repeated observation of breast tumor subtypes in independent gene expression data sets. Proc Natl Acad Sci USA  100: 8418–8423. <a class="find" href="/article/findArticle.action?author=Sorlie&amp;title=Repeated%20observation%20of%20breast%20tumor%20subtypes%20in%20independent%20gene%20expression%20data%20sets.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Pal1" id="pone.0010236-Pal1"></a><span class="authors">Pal S, Datta K, Mukhopadhyay D</span> (2001) Central role of p53 on regulation of vascular permeability factor/vascular endothelial growth factor (VPF/VEGF) expression in mammary carcinoma. Cancer Res  61: 6952–6957. <a class="find" href="/article/findArticle.action?author=Pal&amp;title=Central%20role%20of%20p53%20on%20regulation%20of%20vascular%20permeability%20factor/vascular%20endothelial%20growth%20factor%20(VPF/VEGF)%20expression%20in%20mammary%20carcinoma.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Bian1" id="pone.0010236-Bian1"></a><span class="authors">Bian J, Sun Y</span> (1997) Transcriptional activation by p53 of the human type IV collagenase (gelatinase A or matrix metalloproteinase 2) promoter. Mol Cell Biol  17: 6330–6338. <a class="find" href="/article/findArticle.action?author=Bian&amp;title=Transcriptional%20activation%20by%20p53%20of%20the%20human%20type%20IV%20collagenase%20(gelatinase%20A%20or%20matrix%20metalloproteinase%202)%20promoter.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Wang2" id="pone.0010236-Wang2"></a><span class="authors">Wang SE, Narasanna A, Whitell CW, Wu FY, Friedman DB, et al. </span> (2007) Convergence of p53 and transforming growth factor beta (TGFbeta) signaling on activating expression of the tumor suppressor gene maspin in mammary epithelial cells. J Biol Chem  282: 5661–9. <a class="find" href="/article/findArticle.action?author=Wang&amp;title=Convergence%20of%20p53%20and%20transforming%20growth%20factor%20beta%20(TGFbeta)%20signaling%20on%20activating%20expression%20of%20the%20tumor%20suppressor%20gene%20maspin%20in%20mammary%20epithelial%20cells.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Kunz1" id="pone.0010236-Kunz1"></a><span class="authors">Kunz C, Pebler S, Otte J, von der Ahe D</span> (1995) Differential regulation of plasminogen activator and inhibitor gene transcription by the tumor suppressor p53. Nucleic Acids Res  23: 3710–7. <a class="find" href="/article/findArticle.action?author=Kunz&amp;title=Differential%20regulation%20of%20plasminogen%20activator%20and%20inhibitor%20gene%20transcription%20by%20the%20tumor%20suppressor%20p53.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Teodoro1" id="pone.0010236-Teodoro1"></a><span class="authors">Teodoro JG, Parker AE, Zhu X, Green MR</span> (2006) p53-mediated inhibition of angiogenesis through up-regulation of a collagen prolyl hydroxylase. Science  313: 968–971. <a class="find" href="/article/findArticle.action?author=Teodoro&amp;title=p53-mediated%20inhibition%20of%20angiogenesis%20through%20up-regulation%20of%20a%20collagen%20prolyl%20hydroxylase.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Komarova1" id="pone.0010236-Komarova1"></a><span class="authors">Komarova EA, Krivokrysenko V, Wang K, Neznanov N, Chernov MV, et al. </span> (2005) p53 is a suppressor of inflammatory response in mice. Faseb J  19: 1030–1032. <a class="find" href="/article/findArticle.action?author=Komarova&amp;title=p53%20is%20a%20suppressor%20of%20inflammatory%20response%20in%20mice.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Roger1" id="pone.0010236-Roger1"></a><span class="authors">Roger L, Gadea G, Roux P</span> (2006) Control of cell migration: a tumour suppressor function for p53? Biol Cell  98: 141–152. <a class="find" href="/article/findArticle.action?author=Roger&amp;title=Control%20of%20cell%20migration:%20a%20tumour%20suppressor%20function%20for%20p53?">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Hyder1" id="pone.0010236-Hyder1"></a><span class="authors">Hyder SM, Nawaz Z, Chiappetta C, Stancel GM</span> (2000) Identification of functional estrogen response elements in the gene coding for the potent angiogenic factor vascular endothelial growth factor. Cancer Res  60: 3183–90. <a class="find" href="/article/findArticle.action?author=Hyder&amp;title=Identification%20of%20functional%20estrogen%20response%20elements%20in%20the%20gene%20coding%20for%20the%20potent%20angiogenic%20factor%20vascular%20endothelial%20growth%20factor.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Ferrara2" id="pone.0010236-Ferrara2"></a><span class="authors">Ferrara N, Davis-Smyth T</span> (1997) The biology of vascular endothelial growth factor. Endocr Rev  18: 4–25. <a class="find" href="/article/findArticle.action?author=Ferrara&amp;title=The%20biology%20of%20vascular%20endothelial%20growth%20factor.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Carmeliet1" id="pone.0010236-Carmeliet1"></a><span class="authors">Carmeliet P</span> (2005) Angiogenesis in life, disease and medicine. Nature  438: 932–936. <a class="find" href="/article/findArticle.action?author=Carmeliet&amp;title=Angiogenesis%20in%20life,%20disease%20and%20medicine.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Chappel1" id="pone.0010236-Chappel1"></a><span class="authors">Chappel JC, Taylor SM, Ferrara N, Bautch VL</span> (2009) Local Guidance of Emerging Vessel Sprouts Requires Soluble Flt-1. Developmental Cell  17: 377–386. <a class="find" href="/article/findArticle.action?author=Chappel&amp;title=Local%20Guidance%20of%20Emerging%20Vessel%20Sprouts%20Requires%20Soluble%20Flt-1.">
                    Find this article online
                  </a>
</li>
<li>
<a name="pone.0010236-Menendez4" id="pone.0010236-Menendez4"></a><span class="authors">Menendez D, Inga A, Resnick MA</span> (2006) The biological impact of the human master regulator p53 can be altered by mutations that change the spectrum and expression of its target genes. Mol Cell Biol  26: 2297–2308. <a class="find" href="/article/findArticle.action?author=Menendez&amp;title=The%20biological%20impact%20of%20the%20human%20master%20regulator%20p53%20can%20be%20altered%20by%20mutations%20that%20change%20the%20spectrum%20and%20expression%20of%20its%20target%20genes.">
                    Find this article online
                  </a>
</li>
</ol>
</div>

</div>

</body>
</html>
